Content: New alien species: Incipias (no.3) (#8949)
endless-sky/endless-sky: data/incipias/incipias first contact.txt

				`	Feeling that the situation could escalate any moment, you slowly place the crate with the repair equipment on the ground and walk back a few steps. You are careful not to make any hasty movements. Both sides are still for a moment until one of the aliens slowly floats forward. Tones of yellow are now mixing into the creature's orange color, and two of their tentacles wrap around the crate. The other four back off in response. Once the alien manages to open the crate and look inside, its colors shift to a weak green. The rest of the crew also seem to relax as they take on the same color.`
				`	With the repair equipment delivered there is nothing else you can do, given the communication barrier. Just as you leave you can see how the aliens start pulsating in all colors of the spectrum; gradients are moving over their skin so fast that they almost looks white.`
					goto end
				label flee
				`	As the tension increases even further, you simply drop the repair equipment and run back into your ship, smashing the button that closes the lock. You get a last look on the alien: blue and yellow tones are mixing on their skin, but they are not taking any action.`
				label end

> 				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment.`

Reviewer: Just to make it clear that the stranded ship is now in Turra.
```suggestion
				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment. Instead of leaving the system, however, the ship quietly disappears into the nearest gas giant.`
```

			choice
				`	"Then let me help them."`
					goto initiative
				`	"How can you make such a cold-hearted decision?"`
			`	"This might be difficult for you to understand, but we must place the greater good above individual lives. If we show ourselves to them now by the shock of such an advanced species will fan the flames of chaos."`
			choice
				`	"Then let me help them."`
					goto initiative
				`	"You can deal with the consequences later. Help them now!"`
			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can forsee the likelihood of you making a rash and impetuous decision to contact this ship.`
			choice
				`	"Sure, let me do the work."`
				`	"If you are too weak..."`
			`	Something like a laugh comes from the Quarg, before continuing.`
			label initiative
			`	"You lack the tools for such a mission. To my knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense."`

> 			`	"But on this last point I am able to assist you. I can provide you with a single one of our translators, which will allow you to contact the species in their home system. Request that they sell you one of their ships, so that you are able to land on their planets."`

Author: That was my idea, yes

> 			`	"But help you I can at one point, communication that is. I can provide you with a single one of our translators, which will allow you to contact the species in their home system. To sell you one of their ships ask to them, easy it won't be."`

Reviewer: ```suggestion
			`	"But on this last point I am able to assist you. I can provide you with a single one of our translators, which will allow you to contact the species in their home system. Request that they sell you one of their ships, so that you are able to land on their planets."`
```

				`	"Greetings, Incipias. I am a human."`
				`	"I thank you for your hospitality."`
			`	Movement and colors go through the crowd like waves.`
			`	The 'speaker' continues, "You are the first outer-being that ever communicated with us. Are you the only outer-being out there?"`
			choice
				`	"No, there are others too."`
				`	"The galaxy is full of other beings."`
			`	Another wave goes through the crowd. "So there are more. But, stranger, how can we help you?"`
			choice
				`	(Tell them about the stranded ship and how I want to help them.)`
					goto stranded
				`	"I want one of your ships."`
			`	"We shall allow you to buy one. If that is all that it takes to make you a not-danger. If that is all please excuse us, we need to talk about what this new situation means for us, I hope we can out-question you more another time."`
				goto end
			label stranded
			`	Another wave goes through the crowd but this time you notice an overflow in green tones.`
			`	"So they are still alive. We shall give you access to our ship. Let us prepare the ship-seller. Do notice though that they can not jump between home-places. We hope to question you more on another future-time.`
			label end

> 			`	The five float away but the crowd stays continues staring at you, probably out of interest.`

Reviewer: ```suggestion
			`	The five float away but the crowd continues staring at you, probably out of interest.`
```

			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and the shields must have broke when they ran out of fuel.`
			`	You receive no answer to your hail. Your scanners indicate that it ran out of energy.`
			`	You carefully maneuver your ship closer, until you can dock with the Incipias ship.`
			`	When the lock opens five Incipias float in front of you. Noticing your presence they quickly back off; they were clearly not expecting a human in one of their kind's ships. It takes a moment for them to recognize you.`
			choice
				`	"Hello, I have come to save you."`
				`	"I have been sent by your people to get you home."`
			`	They seem to switch from a tense state to a surprised one.`

> 			`	One of them starts pulsating, "You light our language? How surprising. We would not have survived anothey day here. You are our life-protector. We would love to talk more, but a weather-calamity is coming, we should leave now."`

Reviewer: ```suggestion
			`	One of them starts pulsating, "You light our language? How surprising. We would not have survived another day here. You are our life-protector. We would love to talk more, but a weather-calamity is coming. We should leave now."`
```

> 			`	You tell them to come aboard and launch away from the planet.`

Reviewer: ```suggestion
			`	You tell them to come aboard, and you manage to launch your ship just before a storm hits the area.`

				accept
	on complete
		conversation
			`When you land on the station your passengers thank you again and leave, probably to get some well-earned rest.`

> 			`	As you step out of the <ship> an Incipias approaches you. "Follow me outer-helper before a crowd gathers, we would like to talk."`

Reviewer: ```suggestion
			`	As you step out of the <ship> an Incipias approaches you. "Follow me outer-helper, before a crowd gathers. We would like to talk."`
```

> 			`	They guide you through the hallways so that you avoid larger groups and move only through horizontal corridors. You arrive in a large room with a view over the entire station. Three Incipias are already in the room. One starts glowing.`

Reviewer: ```suggestion
			`	They guide you through the hallways so that you avoid larger groups and move only through horizontal corridors. You arrive in a large room with a view over the entire station, with three Incipias already present. One begins to glow.`
```

> 			`	"Welcome. We are the leaders of the Hicemus, the reigning government of the Incipias. If you agree we would like to ask you a few questions. In return we would be open for questions too."`

Reviewer: ```suggestion
			`	"Welcome. We are the leader-governers of the Hicemus, the reigning government of the Incipias. If you agree we would like to ask you a few question-queries. In return we would be open for question-queries too."`
```

			choice
				`	"Sorry but I have other stuff to do."`
					goto endNoTime
				`	"Sure, ask something."`

> 			`	A green glow comes from all of them, "Thank you for taking the time. Can you tell us more about the deep space surrounding us?"`

Reviewer: ```suggestion
			`	A green glow comes from all of them. "Thank you for taking the time-patience. Can you tell us more about the deep space surrounding us?"`
```

					goto detail
				`	(Give a rough overview of what you have seen.)`
			`	After watching your explanation a rapid-fire sequence of colors are exchanged between them. Your translator cannot follow, so you are left watching them flash in yellow tones mixed with green and red.`
			`	"Thank you for this insight. All we gathered until now was by observing stars. To think that there is so much out there, it truly invokes determination to explore.`
				goto yours
			label detail
			`	After watching your explanation a rapid-fire sequence of colors are exchanged between them. Your translator cannot follow, so you are left watching them flash in yellow tones mixed with green.`
			`	"Thank you for this exhaustive insight. All we gathered until now was by observing stars close to us. To think that there is so much out there, it truly invokes determination to explore.`
			label yours
			`	"But now to you, is there anything you would like to know?"`
			# might add to this list over time
			label choice
			choice
				`	"I do not have any more questions."`
					goto end
				`	"Why do you call this station your capital?"`
					goto capital
				`	"Why can I not land on your home planet?"`

> 			`	A flash of red appears for only a second, "Our origin-world is not directly ruled by us. They may approach you separately at some point in the future, be careful of their lights. They have outdated values long laid off by us, who are progressing fast.`

Reviewer: ```suggestion
			`	A flash of red appears for only a second. "Our origin-world is not directly ruled by us. They may approach you separately at some point in the future, but be careful of their lights. They have outdated values long laid off by us, who are progressing fast.`
```

	invisible
	landing
	to complete
		never
	on enter Il'le
		conversation
			`"You have come to a dead end. Perhaps try another route," a voice with an uncertain origin says.`
			`	Alarmed, you turn around but nobody is there. You check your scanners but there is nothing present in this system except a few indigenous lifeforms. You also notice that your mapping system found hyperlinks stretching out from this system.`
			choice
				`	(Ignore it. It was probably your imagination.)`
				`	"You can't fool me. This is no dead end."`
					goto reason
				`	"Why should I listen to you?"`
					goto reason
			`	"Ignoring me will not help you. Turn around," the same voice says, again with no indication of where it came from.`
			choice
				`	"You can't fool me. This is no dead end."`
				`	"Why should I listen to you?"`
			label reason
			`	"Here lie the unborn flowers yet to come into full bloom. There is nothing here for you to find. Listen to my advice and turn back." The voice sounds more energetic this time and you feel as if it is coming out of your head.`
			choice
				`	"Unborn flowers. That sounds interesting."`
				`	"I will see for myself."`

> 			`	A sigh seems to drift through the continuum, "Human, these unborn flowers-" an impression of euphoria comes over you, "-would only be endangered by your presence. Please respect the beauty of life and do not step into this region that will soon bring forth a new species. Your kind is much more advanced than anything that you could hope to find here. No treasures here await you."`

Reviewer 2: But this Archon only has a hyperdrive, so it should probably be different in other ways as well, correct?

> 			`	A sigh seems to drift through the continuum, "Human, these unborn flowers-" an impression of euphoria comes over you, "-would only be endangered by your presence. Please respect the beauty of life and do not step into this region that will soon bring forth a new species. Your kind is much more advanced than anything that you could hope to find here. No treasures here await you."`

Author: The hyperdrive is more of an internal thing than something that is present in game, the hyperdrive makes the archon stay in the system group while still following the player.

If the writing style for the archons is generally as you said Arachi I can totally change it to that. But I have to say that I find the conversation more fluid this way.

				to display
					not "Incipias: Meet the Quarg: offered"
			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. This must be the species the Quarg talked about.`
				to display
					has "Incipias: Meet the Quarg: offered"

mission "Incipias: Meet the Quarg"
	invisible
	landing
	source
		government "Quarg (Incipias)"
	to complete
		never
	on offer
		conversation
			branch "ut divitas" "spec inci"
				has "flagship planet: Ut Divitas"
			label "ut divitas"
			`Your spaceship lands with a soft jolt. A small group of Quarg quickly gathers around your ship, making some simple but clear gestures telling you to wait. After they've realized that you understood, one of them walks away in a hurried manner.`
			`	Only a moment later the Quarg returns with another Quarg. The new one starts talking with a very strange accent.`
				goto "both planets"
			label "spec inci"

> 			`	You only wait for a few minutes before three Quarg appear on the landing pad. The one in the middle is wearing clothing that makes clear that it is a military official. The two in the back are wearing full combat gear but do not seem to expect a fight. The one in the middle starts talking in a very strange accent.`

Reviewer 3: Probably should make the described waiting periods far longer, closer to the case with the Gegno Quarg finding someone who can speak human. There you go and are moved around the ring a bunch, while here I imagine it would be more the case you just wait for many/a few hours depending on the planet, as the Quarg scramble to get someone available there that can communicate with you.

The reasoning for this is that the Quarg here are far less resourceful than what would be the norm for them, not having a ring at the ready with all the tech there, and whatever teleporty thingies they may have probably wouldn't be as ready to go as in cases where they have a ringworld (assuming this Quarg comes in via some magic Quarg TP machines, which I assume as there just being a Quarg around who happens to speak human in a desolate region by Quarg-standards seemed less likely).

For the Spec Inci one, I'd also remove the mention of the distinctive garb for the Quarg that speaks, as it's a point made throughout the game that the player isn't really ever sure of what Quarg can do what, as seen in Wanderers when you seemingly just happen upon a Quarg who somehow has the authority to send 3 Wardragons to guard the Wanderers, despite the same conversation noting that `"The Quarg do not generally share any information on how their leadership hierarchy works, if such a hierarchy even exists"`.

Author: I increased the waiting periods and removed the garb

				`	With the repair equipment delivered there is nothing else you can do, given the communication barrier. Just as you leave you can see how the aliens start pulsating in all colors of the spectrum; gradients are moving over their skin so fast that they almost look white.`
					goto end
				label flee
				`	As the tension increases even further, you simply drop the repair equipment and run back into your ship, smashing the button that closes the lock. You get a last look on the alien: blue and yellow tones are mixing on their skin, but they are not taking any action.`
				label end
				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment. Instead of leaving the system, however, the ship quietly disappears into the nearest gas giant.`

> 				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen. Notably this ship had no hyperdrive or anything similar installed; strange considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`

Reviewer 2: Does the player have an Outfit Scanner? If not, they probably shouldn’t be able to see that the Incipia ship has no hyperdrive.

Reviewer 2: Of course, that would require losing a lot of information about the Incipias, so I’m not sure how that should be handled.

Author: Ye I will need to think about this one, maybe hyperdrives are just obvious?

Reviewer 2: Or else when you dock with something it automatically transfers information about it? That would explain a lot of things.

Reviewer 2: @Amazinite would the above be correct?

				action
					set "incipias: helped the derelict"
		personality
			confusion 50
			derelict uninterested staying
	on enter "Hui'uc"
		conversation

> 			`On entering the system you find an alien spaceship disabled and badly damaged. There does not appear to be any other ships or inhabited planets in the system.`

Reviewer 2: ```suggestion
			`On entering the system you find an alien spaceship disabled and badly damaged. There do not appear to be any other ships or inhabited planets in the system.`
```
Doing this way might be better.

mission "Incipias: Help the stranded Part 1"
	landing
	source
		government "Quarg (Incipias)"
	destination "Pon'tes"
	to offer
		has "incipias: helped the derelict"
	on offer
		conversation

> 			`Right after you arrived on the planet you go looking for a Quarg official to tell about the stranded ship. Luckily it is not long before you find one.`

Reviewer 3: Again on the hierarchy thing, maybe just have it that you look for the first Quarg who can speak your language?

Author: did that

			`	It looks at you curiously. "So you met them? I am surprised that you were able to communicate. It took a very long time even for us to decipher their way of communication."`
			`	You quickly explain what happened. The Quarg seems very relieved about the fact that you were not able to talk to them.`
			label explanation
			`	"You ran into a most peculiar situation. This species should not be able to cross the borders of their system yet." It tilts it head for a moment considering what you just said. "It is our desire to help them, but it is too early for us to reveal ourselves to their species. I fear there is nothing we can do." It shakes its head.`
			choice
				`	"Then let me help them."`
					goto initiative
				`	"How can you make such a cold-hearted decision?"`
			`	"This might be difficult for you to understand, but we must place the greater good above individual lives. If we show ourselves to them now, the shock of such an advanced species appearing in their space will fan the flames of chaos."`
			choice
				`	"Then let me help them."`
					goto initiative
				`	"You can deal with the consequences later. Help them now!"`

> 			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can forsee the likelihood of you making a rash and impetuous decision to contact this ship."`

Reviewer 4: ```suggestion
			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can foresee the likelihood of you making a rash and impetuous decision to contact this ship."`
```
typo

			choice
				`	"Sure, let me do the work."`
				`	"If you are too weak..."`

> 			`	Something like a laugh comes from the Quarg, before continuing.`

Reviewer 3: If you mean for it to be the Quarg laughing, their equivalent of that appears to be clicking their teeth together rapidly. If you meant they make something akin to human laughter, which'd mean another reaction altogether, just ignore this comment.

Author: yea I meant the quarg to laugh, so I changed it to clicking

			label initiative
			`	"You lack the tools for such a mission. To my knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense."`
			`	"But for the sake of the Incipias, I will assist you on this last point. If you cannot be dissuaded from your course of action, then it is best you are given a means of communication, so that your interactions can at least be on amicable terms. I can provide you with a single one of our translators, which will allow you to contact the species in their home system. Request that they sell you one of their ships, so that you are able to land on their planets."`
			`	It pauses before taking on a more serious tone. "However, you must remember one thing. You are forbidden from granting them any advanced technology of yours. If you agree to that point we shall assist you."`
			label choice
			choice
				`	"If that's how I can help them, sure."`
					goto yes
				`	"Yes, but why do you not want me to sell my tech?"`

> 			`	"Halting the natural increase of a species is unpredictable at the best of times. Your species lacks the wisdom to understand this, as you are young, and have not seen what we have seen." The Quarg pauses for a moment, lost in thought. "The foreseeable risks of even letting you contact them are monumental, and not something you should consider trivial."`

Reviewer 2: ```suggestion
			`	"Speeding up the natural increase of a species is unpredictable at the best of times. Your species lacks the wisdom to understand this, as you are young, and have not seen what we have seen." The Quarg pauses for a moment, lost in thought. "The foreseeable risks of even letting you contact them are monumental, and not something you should consider trivial."`
```
The player would not be halting the increase if they granted technology.

Author: yes but they would halt the _natural_ increase

Reviewer 2: Okay.

				accept
	on enter Dom'us
		"reputation: Hicemus" += 10
		conversation
			`Entering the system with the translator, you prepare to broadcast your first message.`
			choice
				`	"Hello. I am Captain <first> <last>, here to make contact."`
				`	"Greetings, strangers. I seek to help you."`
				`	"I come in peace."`
			`	You notice that your translator starts to flash a sequence of lights, and you point it towards your ship's camera. For a minute, nothing happens at all.`
			`	After some time, a wave of movement passes through all ships in the system, and a communication channel opens from the space station. Colors rush over your display: a mix of yellow and green. You point your translator towards the flashing lights and hope it is able to pick up whatever these aliens are saying to you.`
			`	"Welcome world-traveler. We greet you in our home-space."`
			`	The colors stop before rushing over your screen again.`
			`	"What is your intention-goal here?"`
			choice
				`	"I am here to make peaceful contact with you."`
				`	"I want to help one of your stranded ships."`

> 			`	Again a long pause. "Your intention-goals are welcome. Please dock to capital-station."`

Reviewer 2: ```suggestion
			`	Again a long pause. "Your intention-goals are welcome. Please dock to the capital-station."`
```
The? Or is it a name?

	on complete
		conversation
			`Thanks to the map you are easily able to dock with the station. Your airlock opens and you enter an environment like nothing you ever saw before. There are no visible pathways, but instead foggy hallways, some of which even ascend vertically. A crowd of beings gathered in front of your airlock each flash a multitude of colors. A group of five floats in front of the crowd, and the middle one starts pulsating in calm tones.`
			`	"Welcome, outer-being, to our capital. We are the Incipias."`
			`	It stops as if awaiting an answer.`
			choice
				`	"Greetings, Incipias. I am a human."`
				`	"I thank you for your hospitality."`
			`	Your translator begins rapidly flashing. The Incipias seem to respond to this - movement and colors begin to diffuse through the crowd like waves.`
			`	The 'speaker' continues, "You are the first outer-being that ever communicated with us. Are you the only outer-being out there?"`
			choice
				`	"No, there are others too."`
				`	"The galaxy is full of other beings."`
			`	Another wave goes through the crowd. "So there are more. But, stranger, how can we help you?"`
			choice

> 				`	(Tell them about the stranded ship and how I want to help them.)`

Reviewer 2: Should this, and the one earlier, be went?

Author: no you only provided some assistance, they still need help to return

Reviewer 2: You can mark it as resolved.

				goto end
			label stranded
			`	Another wave goes through the crowd but this time you notice an overflow in green tones.`
			`	"So they are still alive. We shall give you access to our ship. Let us prepare the ship-seller. Do note though that they can not jump between system-places, not like your own ship. We hope to question you more on another future-time.`
			label end
			`	The five float away but the crowd continues staring at you, probably out of interest.`

mission "Incipias: Help the stranded Part 2"
	landing
	passengers 5
	source
		planet Turra
	destination "Pon'tes"
	to offer
		has "Incipias: Help the stranded Part 1: done"
	on offer
		conversation
			`Your ship dives into the clouds of the gas giant, an environment on a scale you never experienced before. Giant clouds tower like castles, sometimes opening to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`

> 			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and the shields must have broke when they ran out of fuel.`

Reviewer 2: Shields broke? I thought the Incipias had energy-based shields like other species do.

Author: there is not much information on how the shields in ES work, the incipias shields asume that there is some sort of long lasting fuel that would normally never run out, but in this case, it ran out, maybe because of damage on the ship

Reviewer 2: It was mentioned how the Puffin was the only ship that would work for Face to Maw because of its old-type metal shields. But the wiki says [here](https://github.com/endless-sky/endless-sky/wiki/PlayersManual#outfitting-your-ship)
> hull will include an integrated energy shield matrix
However, that is for human ships, and the Incipias might be different. Your choice.

Reviewer 2: @RisingLeaf there is this.

Author: again what I mean is not that the shields themself broke but the shield generator, I dont think I made that clear

Reviewer 2: If that is made clear it would help a lot… although that also runs into the question how do you know? Maybe because the ship doesn’t have shields?

Reviewer 2: ```suggestion
			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and the shield generator must have broke when they ran out of fuel, because their shields are completely down.`
```
Something like this?

			`	You receive no answer to your hail. Your scanners indicate that it ran out of energy.`
			`	You carefully maneuver your ship closer, until you can dock with the Incipias ship.`
			`	When the lock opens five Incipias float in front of you. Noticing your presence they quickly back off; they were clearly not expecting a human in one of their kind's ships. It takes a moment for them to recognize you.`
			choice
				`	"Hello, I have come to save you."`
				`	"I have been sent by your people to get you home."`
			`	They seem to switch from a tense state to a surprised one.`

> 			`	One of them starts pulsating, "You light our language? How surprising. We would not have survived another day here. You are our life-protector. We would love to talk more, but a weather-calamity is coming. We should leave now."`

Reviewer 2: Light our language feels a little hard to understand, even though it is correct from a literal point of view.

Author: Well thats their quirk, i dont think it would help immersion to switch to talk

Reviewer 2: Mark it as resolved than.

				accept
	on complete
		conversation
			`When you land on the station your passengers thank you again and leave, probably to get some well-earned rest.`
			`	As you step out of the <ship> an Incipias approaches you. "Follow me [stranger, helper] before a crowd gathers, we would like to talk."`
			`	It guides you through the hallways avoiding larger groups, only moving through horizontal corridors. You arrive in a large room with a view over the entire station. Three Incipias are already in the room. One starts glowing.`
			`	"Welcome. We are the leaders of the Hicemus, the [reigning, most powerful] government of the Incipias. If you agree we would like to ask you a few questions. In return we would be open for questions too."`
			choice
				`	"Sorry but I have other stuff to do."`
					goto endNoTime
				`	"Sure, ask something."`
			`	A green glow comes from all of them, "Thank you for taking the time. Can you tell us more about the [unknown territory, galaxy] surrounding us?"`
			choice
				`	(Give them an exhausting story of what you have seen so far.)`
					goto detail
				`	(Give a rough overview of what you have seen.)`

> 			`	After watching your explanation a quickfire of colors that your translator can not follow is exchanged between them, you mostly see yellow tones mixed with green and red.`

Reviewer: ```suggestion
			`	After watching your explanation a rapid-fire sequence of colors are exchanged between them. Your translator cannot follow, so you are left watching them flash in yellow tones mixed with green and red.`
```

> 				`	"Sure, ask something."`

Reviewer: ```suggestion
				`	"Sure, ask me something."`
```

			`	"Thank you for this insight. All we [knew, gathered] until now was by observing stars. To think that there is so much out there, it truly invokes determination to explore.`
				goto yours
			label detail

> 			`	After watching your story a quickfire of colors that your translator can not follow is exchanged between them, you mostly see yellow tones mixed with green.`

Reviewer: ```suggestion
			`	After watching your explanation a rapid-fire sequence of colors are exchanged between them. Your translator cannot follow, so you are left watching them flash in yellow tones mixed with green.`
```

> 			`	After watching your explanation a rapid-fire sequence of colors are exchanged between them. Your translator cannot follow, so you are left watching them flash in yellow tones mixed with green and red.`

Reviewer: ```suggestion
			`	After seeing your explanation a rapid-fire sequence of colors are exchanged between them. Your translator cannot follow, so you are left watching them flash in yellow tones mixed with green and red.`
```

> 			`	"Thank you for this insight. All we gathered until now was by observing stars. To think that there is so much out there, it truly invokes determination to explore.`

Reviewer: ```suggestion
			`	"Thank you for this insight. All we gathered until now was by observing-studying stars. To think that there is so much out there... it truly invokes a determination to explore.`
```

> 			`	"Thank you for this exhaustive insight. All we gathered until now was by observing stars close to us. To think that there is so much out there, it truly invokes determination to explore.`

Reviewer: ```suggestion
			`	"Thank you for this exhaustive insight. All we gathered until now was by observing-studying stars close to us. To think that there is so much out there... it truly invokes a determination to explore.`
```

			label yours

> 			`	"But now to you, is there anything you would like to know?"`

Reviewer: ```suggestion
			`	"But now to you. Is there anything you would like to know?"`
```

			# might add to this list over time
			label choice
			choice
				`	"I do not have any more questions."`
					goto end
				`	"Why do you call this station your capital?"`
					goto capital
				`	"Why can I not land on your home planet?"`
			`	A flash of red appears for only a second. "Our origin-world is not directly ruled by us. They may approach you separately at some point in the future, but be careful of their lights. They have outdated values long laid off by us, who are progressing fast.`
				goto choice
			label capital
			`	The one pulsating now takes on a bright green color, "This station is our greatest achievement, our star-bridge. In honor of our urge to expand we made this station our capital to demonstrate our determination."`
				goto choice
			label end

> 			`	"Thank you again for your time. You have given us a lot to process. We hope you too have benefited from this meeting-talk."`

Reviewer: ```suggestion
			`	"Thank you again for your time. You have given us much to process. We hope you too have benefited from this meeting-talk."`
```

#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Incipias: Guardian of Incipias Space"
	invisible
	landing
	repeat
	to offer
		not "Incipias: Guardian of Incipias Space: active"
	to complete
		never
	npc save
		government "Drak (Incipias)"
		personality heroic vindictive frugal forbearing
		system Il'le
		ship "Archon (Hyperdrive Only)" "Guardian of Spring"

mission "Incipias: Warning of the Guardian"
	invisible
	landing
	to complete
		never
	on enter Il'le
		conversation
			`"You came to a dead end, maybe try another route," a voice with uncertain origin says.`

> 			`	Alarmed by the sudden appearence you turn around but nobody is there. You check your scanners but only a few indigenous lifeforms that showed no sign of communication appear. Also you notice that your mapping system found hyperlinks stretching out from this system.`

Author: ```suggestion
			`	Alarmed by the sudden appearance you turn around but nobody is there. You check your scanners but only a few indigenous lifeforms that showed no sign of communication appear. Also you notice that your mapping system found hyperlinks stretching out from this system.`
```

> 			`	Alarmed by the sudden appearance you turn around but nobody is there. You check your scanners but only a few indigenous lifeforms that showed no sign of communication appear. Also you notice that your mapping system found hyperlinks stretching out from this system.`

Author: why remove the alarmed?

			choice
				`	(It was probably your imagiation, ignore it.)`
				`	"You cant fool me, this is no dead end."`
					goto reason
				`	"Why should I listen to you?"`
					goto reason
			`	"Ignoring me wont help you, turn around," the same voice says, again with no indication of where it came from.`
			choice
				`	"You cant fool me, this is no dead end."`
				`	"Why should I listen to you?"`
			label reason
			`	"Here lie the unborn flowers yet to come to full bloom. There is nothing here for you to find, listen to my advice and turn back." The voice sounds more energetic this time and you get the impression that is coming out of your head.`

> 			choice

Author: i do not know how to implement that though, for example you accept, but what stops you from flying further?

> 			choice

Author: There should be a follow up to that then, right? the drak would not be happy.

> 			choice

Author: I made it so that you can accept the warning and added a small follow up if you deceive the drak

				`	The voice fades out leaving nothing behind but the feeling of being watched.`
				decline

mission "Incipias: First Contact I"
	invisible
	landing
	to complete
		never
	on enter Dom'us
		conversation
			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. Maybe you can find some more information in nearby systems.`
				to display
					not "Incipias: Meet the Quarg: offered"
			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. This must be the species the Quarg talked about.`
				to display
					has "Incipias: Meet the Quarg: offered"

mission "Incipias: Meet the Quarg"
	invisible
	landing
	source
		government "Quarg (Incipias)"
	to complete
		never
	on offer
		conversation

> 			branch "ut divitas" "spec inci"

Author: yes:
```html
branch <if true> [<if false>]
		<condition> <comp> <value>
		(has | not) <condition>
		(and | or)
```

				has "flagship planet: Ut Divitas"
			label "ut divitas"

> 			`Your spaceship lands with a soft jolt. A small group of Quarg quickly gathers around your ship, making some simple but clear gestures telling you to wait. After they noticed that you understood them, one of them walks away in a hurried manner.`

Author: ok I am no expert in english, so excuse this question, group is third person singular, right? so doesnt it need the 's'

			`	Only a moment later the Quarg returns with another Quarg. The new one starts talking with a very strange accent.`
				goto "both planets"
			label "spec inci"

> 			`You land on one of the automated landing pads. As soon as you shut your engines off it starts to move into a hangar slot. The gates of the hangar close. You leave your ship and notice a door on the side of the hangar, a sign there is showing something in a foreign language, but as you get closer it turns white for a moment, before the words "HUMAN DETECTED, PLEASE WAIT WHILE ASSISTANCE IS ON THE WAY" appear. Left with no other choice you have to stay where you are.`

Author: I see where you are coming from, will remove it

			`	Luckily it only takes a few minutes until the doors open and three Quarg appear behind it. The one in the middle is wearing clothing that makes clear that it is a military official. The two in the back are wearing full combat gear but do not seem to expect a fight. The one in the middle starts talking in a very strange accent.`
			label "both planets"
			`	"Welcome human, long ways came you. Kind of yours never here have been, your mission?"`
			choice
				`	"I am just exploring the galaxy."`
					goto "explorer"
				`	"I am looking for new species."`
					goto "new species"
			label "explorer"

> 			`	"An obvious pursuit for the young that is," it nods, "if advice you I may, to not walk this way in a careless manner, the galaxy holds many dangers that your kind has yet to discover or even to understand." It straightens up to its full height, towering over you, "but in the end worth the trouble it will be."`

Author: the work will pay of

				goto "end"
			label "new species"

> 			`	"What a peculiar task that is," it looks confused and tilts its head, "what to achieve by this you hope? That an incredible amount of dangers come with meeting other species you not know? For a reason invisible borders in space there are. To be careful and consider every last step I advise."`

Author: true, what would be a good alternative that also sounds broken?

			`	With these words the Quarg turns around and leaves. It seems like you are free to use the local facilities now.`

# Maybe add a version of the ships sprite that looks damaged/modded.
ship "Nimbus" "Nimbus FTL"
	add attributes
		"bunks" -25
	outfits
		"Liquid-Class Shield"
		"Metallic Hydrogen Cell"
		"Cloud Piercer" 20

		"MH Red-Class Engine"
		"MH Red-Class Steering"

mission "Incipias: The Derelict"
	landing
	invisible
	to offer
		has "Incipias: Meet the Quarg: offered"
	npc assist
		government "Hicemus"
		ship "Nimbus FTL" "HIC Veni #000001"
		system "Hui'uc"
		to despawn
			has "incipias: helped the derelict"
		on assist
			conversation
				`When you approach the ship you get hailed and as you accept the hail color gradients start moving over your screen but your speakers stay silent. It seems like communication over these channels is doomed to fail, you continue your boarding maneuver.`
				scene "scene/incipias message"

> 				`	You manage to dock to the disabled ship quite easily. The lock opens and a hallway opens up before you, obviously not built for humans. The walls are strangely curved and the floor has nearly no grip at all. Your view gets caught by the five giant aliens half octopode, half jellyfish that are pulsating in bright colors, similar to the gradients that appeared on your screen earlier. One of them is floating behind a giant mounted gun, the two barrels are pointed right at you. The gunner flashes in an orange light, and you can feel the tension in the room rising.`

Reviewer: ```suggestion
				`	You manage to dock to the disabled ship quite easily. The lock opens and a hallway opens up before you. It is immediately obvious that this ship was not designed for humans; the walls are strangely curved and the floor has nearly no grip at all. Your view gets caught by five giant aliens, half octopode, half jellyfish in appearance. They pulsate in bright colors similar to the gradients that appeared on your screen earlier. One of them is floating behind a giant mounted gun, and the two barrels are pointed right at you. The gunner flashes in an orange light, and you can feel the tension in the room rising.`
```

				choice
					`	(Try to convey that you intend no harm.)`
						goto peace
					`	(Drop the repair equipment and run back into your ship.)`
						goto flee
					`	(Try to make a non-lethal shot at the one behind the turret and run!)`

> 				`	You hit the creature manning the turret, causing it to explode with a strident sound. Before you can enjoy your short-lived victory, the other turret aims your way. Turning your head you see the gunner flashing in a deep red, before shooting you down, avenging their fallen comrade.`

Reviewer: Where did the second turret come from?

Author: it got lost when rephrasing 😆 , will fix that

					die
				label peace

> 				`	Feeling that the situation could escalate every moment, you slowly place the crate with the repair equipment on the ground, careful to not make any hasty movements and walk back a few steps. No side moves for a moment until one of the aliens slowly moves forward, tones of yellow are now mixing into the orange. Two of their tentacles wrap around the crate, the other four are backing of a little bit. The front one succeeds in finding the opening mechanism, right after it got a look inside, its colors shift to a weak green. The other four also seem to relax as they take on the same color.`

Reviewer: ```suggestion
				`	Feeling that the situation could escalate any moment, you slowly place the crate with the repair equipment on the ground and walk back a few steps. You are careful not to make any hasty movements. No side moves for a moment until one of the aliens slowly floats forward. Tones of yellow are now mixing into the creature's orange color, and two of their tentacles wrap around the crate. The other four back off. Once the alien manages to open the crate and look inside, its colors shift to a weak green. The rest of the crew also seem to relax as they take on the same color.`
```

> 				`	With the repair equipment delivered there is nothing else you can do and communication is not working so you turn around and leave. Just as you leave you can see how the aliens start pulsating in all colors of the spectrum, gradients are moving over their skin so fast that it almost looks white.`

Reviewer: ```suggestion
				`	With the repair equipment delivered there is nothing else you can do, given the communication barrier. Just as you leave you can see how the aliens start pulsating in all colors of the spectrum; gradients are moving over their skin so fast that it almost looks white.`
```

					goto end
				label flee

> 				`	As the tension starts raising even further, you just drop the repair equipment and run back into your ship, you smash the button that closes the lock. You get a last look on the alien, blue and yellow tones are mixing on their skin, but they are not taking any action.`

Reviewer: ```suggestion
				`	As the tension starts raising even further, you simply drop the repair equipment and run back into your ship, smashing the button that closes the lock. You get a last look on the alien: blue and yellow tones are mixing on their skin, but they are not taking any action.`
```

				label end
				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment.`
				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen. Notably this ship had no hyperdrive or anything similar installed; strange considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`
				action
					set "incipias: helped the derelict"
		personality
			confusion 50
			derelict uninterested staying
	on enter "Hui'uc"
		conversation

> 			`On entering the system you find an alien spaceship disabled and badly damaged. You remember your training as a captain when your teacher told you about something about a codex of captains, "A captain never leaves a ship in need behind."`

Reviewer: ```suggestion
			`On entering the system you find an alien spaceship disabled and badly damaged. There does not appear to be any other ships or inhabited planets in the system.` 
```

mission "Incipias: Help the stranded Part 1"
	landing
	source
		government "Quarg (Incipias)"
	destination "Pon'tes"
	to offer
		has "incipias: helped the derelict"
	on offer
		conversation
			`Right after you arrived on the planet you go looking for a Quarg official to tell about the stranded ship. Luckily it does not take long before you find one.`

> 			`	"Human, in quite a haste you seem to be. What the so pressing matter could be?" It asks as it clearly recognizes that you are a bit out of breath.`

Reviewer: I don't feel like the last bit is necessary.
```suggestion
			`	"Human, in quite a haste you seem to be. What the so pressing matter could be?" it asks.`
```

			choice
				`	(Explain the situation.)`
					goto explanation
				`	"I need advice regarding the species in this region."`

> 			`	It looks at you curiously, "So you met them? I am surprised that you were able to communicate, it took a very long time even for us to decipher their way of communication."`

Reviewer: ```suggestion
			`	It looks at you curiously, "So you met them? I am surprised that you were able to communicate. It took a very long time even for us to decipher their way of communication."`
```

			`	You quickly explain what happened and the Quarg seems very relieved about the fact that you were not able to talk to them, it probably was a big achievement for them.`
			label explanation
			`	"A most peculiar situation you ran into, should not be able to cross the borders of their system yet this species..." It tilts it head for a moment considering what you just said. "To help them I would love, but no ships we have that on gas giants can land and too early it is for us to reveal ourselves to them, I fear nothing there is we can do." It shakes its head.`

> 			choice

Author: Would make sense, but it would cancel all contact to the incipias?

> 			choice

Reviewer 3: Perhaps have such an option send you to the `	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can foresee the likelihood of you making a rash and impetuous decision to contact this ship."` part, or something similar?

> 			choice

Author: I added an option to cancel the first contact

				`	"Then let me help them."`
					goto initiative
				`	"How can you make such a cold-hearted decision?"`
			`	"This difficult for you to understand might be, but the greater good over the sole life I have to place. If ourselves show to them now by the shock of such and advanced species existing will be created chaos."`
			choice
				`	"Then let me help them."`
					goto initiative
				`	"You can deal with the consequences later, help them now!"`
			`	"Insisting you sure are human, understand I can your point but intervention by us out of question it is. But a proposal for you I have, to you, a solo captain the means to help I will grant, of course some conditions there are."`
			choice
				`	"Sure let me do the work."`
				`	"If you are too weak..."`

> 			`	Something like a giggle comes from the Quarg, before continuing.`

Author: ```suggestion
			`	Something like a laugh comes from the Quarg, before continuing.`
```
do they laugh?

			label initiative

> 			`	"Your initiative most welcome is human, but not the tools for such a mission you have. To my knowledge no vessels capable of landing on gas giants ever developed by your kind. And as said you, not capable of communicating with them you are."`

Author: sounds good

			`	"Happy I am that you agree to that condition. Please for a moment wait here," it says and walks away.`
			`	It only takes it a moment to get back carrying a sleek silver box with a display on one side.`
			`	"Our translator this is. Into your language it can translate thanks to our database. Be aware of the honor of me handing this you," it says, handing the translator over.`
			`	As you stow it away it explains how to use it. "Oh and captain one last thing, the complexity of their language even we can not express, the translator might not be able to translate some parts directly, when it happens you will notice."`
			`	The quarg wishes you best of luck and leaves you alone. Time to head to Dom'us.`
				accept
	on enter Dom'us
		"reputation: Hicemus" += 10
		conversation
			`Entering the system with the translator, you prepare to broadcast your first message.`
			choice
				`	"Hello. I am captain <first> <last> here to make contact."`
				`	"Greetings, strangers. I seek to help you."`
				# silly option will probably remove it but for now it is just funny
				`	"Howdy aliens, I come in peace.`
			`	For a minute, nothing happens at all, it seems like a communication went off.`

> 			`	After some time, a motion goes through all ships in the system and then a channel opens from the space station. Colors rush over your display a mix of yellow and green, and you point the translator to the display as the Quarg told you.`

Reviewer: ```suggestion
			`	After some time, a wave of movement passes through all ships in the system, and a communication channel opens from the space station. Colors rush over your display: a mix of yellow and green. You point your translator towards the flashing lights and hope it is able to pick up whatever the Incipias are saying to you.`
```

# Copyright (c) 2023 by RisingLeaf (https://github.com/RisingLeaf)
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Incipias: Guardian of Incipias Space"
	invisible
	landing
	repeat
	to offer
		not "Incipias: Guardian of Incipias Space: active"
	to complete
		never
	npc save
		government "Drak (Incipias)"
		personality heroic vindictive frugal forbearing
		system Il'le
		ship "Archon (Hyperdrive Only)" "Guardian of Spring"

> mission "Incipias: Warning of the Guardian"

Reviewer 5: This feels too straightforward. Also, it's wildly different from how Archons are known to behave.

Reviewer: Agreed. The Drak are not this bombastic and aggressive. I'd tone it down a little and keep it more in line with the Conversation on Sayaiban mission.

Author: updated the drak convo, to make it more calm and the style a bit more poetic?
please have a look

	invisible
	landing
	to complete
		never
	on enter Il'le
		conversation
			`A presence sweeps over you leaving behind a sense of otherworldly power. A voice starts to reverberate in your mind, "You have entered space that is not meant for humans, leave!"`
			`	A quick look on your scanner tells you that only a few indigenous lifeforms are in the system.`
			choice
				`	"Who are you?"`
					goto who

> 				`	"I am just a traveler please let me pass."`

Reviewer 5: ```suggestion
				`	"I am just a traveler, please let me pass."`
```

					goto reasonable
				`	"You have no right to command me."`

> 			`	You suddenly get the feeling of your ship starting to shake, the stars in your window start to emit an unatural bright blue light, blinding you. "You better not challenge me human. There are powers in space your kind wont understand for millenias, arrogance will lead to your downfall."`

Reviewer 5: ```suggestion
			`	You suddenly get the feeling of your ship starting to shake, the stars in your window start to emit an unnatural bright blue light, blinding you. "You better not challenge me human. There are powers in space your kind won't understand for millennia, arrogance will lead to your downfall."`
```

			`	The voice pauses and continues more silently "...or until their downfall. Hence why I would prefer you to turn around, human."`
			choice
				`	"I promise I mean no harm."`
					goto reasonable
				`	"Can I not be a good influence?"`
			`	"A cooperation between species is not unheard of, but the opposite is the more common fate for such a relation. I understand your goodwill and I do not want to destroy it." The voice disappears for a moment.`
			`	"Well human it seems like I can not move you to leave this space, I will not keep you from passing through, but I will keep an eye on you."`
				goto end
			label reasonable
			`	"Your confusion is understandable," the voice says with an annoyed tone, "your kind always wanted to expand. I will let you pass but be aware that I am watching you and should you come to any stupid idea I will end your wrong doing."`
			label end
			`	The presence fades leaving no sign of it ever talking to you.`
				decline

mission "Incipias: First Contact I"
	invisible
	landing
	to complete
		never
	on enter Dom'us

> 		conversation

Reviewer 5: Maybe this would be better as a dialog (if that supports `to display`).

Author: they dont support to display and while it could be added, I feel dialog should have a very simple syntax, it is just a notification
What I could do is create two separate missions

			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. Maybe you can find some more information in nearby systems.`
				to display
					not "Incipias: Meet the Quarg: offered"
			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. This must be the species the Quarg talked about.`
				to display
					has "Incipias: Meet the Quarg: offered"

mission "Incipias: Meet the Quarg"
	invisible
	landing
	source
		government "Quarg (Incipias)"
	to complete
		never
	on offer
		conversation
			branch "ut divitas" "spec inci"
				has "flagship planet: Ut Divitas"
			label "ut divitas"

> 			`With a soft jolt your spaceship lands. A small group of Quarg quickly gathers around your ship, they make some simple but clear gestures to tell you to wait. After they noticed that you understood them. One of them walks away in a hurrily manner.`

Reviewer 5: ```suggestion
			`With a soft jolt your spaceship lands. A small group of Quarg quickly gathers around your ship, they make some simple but clear gestures to tell you to wait. After they noticed that you understood them. One of them walks away in a hurried manner.`
```

> 			`With a soft jolt your spaceship lands. A small group of Quarg quickly gathers around your ship, they make some simple but clear gestures to tell you to wait. After they noticed that you understood them. One of them walks away in a hurried manner.`

Reviewer 5: ```suggestion
			`Your spaceship lands with a soft jolt. A small group of Quarg quickly gathers around your ship, making some simple but clear gestures telling you to wait. After they noticed that you understood them, one of them walks away in a hurried manner.`
```

			`	Only a moment later the Quarg returns with another Quarg. The new one starts talking with a very strange accent.`
				goto "both planets"
			label "spec inci"

> 			`You land on one of the automised landing pads. As soons as you shut your engines off it starts to move into a hangar slot. The gates of the hangar close. You leave your ship and notice a door on the side of the hangar, a sign there is showing something in a foreign language, but as you get closer it turns white for a moment, before the words "HUMAN DETECTED, PLEASE WAIT WHILE ASSISTANCE IS ON THE WAY" appear. Left with no other choice you have to stay where you are.`

Reviewer 5: ```suggestion
			`You land on one of the automated landing pads. As soon as you shut your engines off it starts to move into a hangar slot. The gates of the hangar close. You leave your ship and notice a door on the side of the hangar, a sign there is showing something in a foreign language, but as you get closer it turns white for a moment, before the words "HUMAN DETECTED, PLEASE WAIT WHILE ASSISTANCE IS ON THE WAY" appear. Left with no other choice you have to stay where you are.`
```
~~I'm guessing you don't mean atomized.~~

			`	Luckily it only takes a few minutes until the doors open and three Quarg appear behind it. The one in the middle is wearing clothing that makes clear that it is a military official. The two in the back are wearing full combat gear but do not seem to expect a fight. The one in the middle starts talking in a very strange accent.`
			label "both planets"
			`	"Welcome human, long ways came you. Kind of yours never here have been, your mission?"`
			choice
				`"I am just exploring the galaxy."`
					goto "explorer"
				`"I am looking for new species."`
					goto "new species"
			label "explorer"

> 			`	"An obvious pursuit for the young that is," it nods, "if advice you I may, to not walk this way in a careless maner, the galaxy holds many dangers that your kind has yet to discover or even to understand." It straightens up to its full height, towering over you, "but in the end worth the trouble it will be."`

Reviewer 5: ```suggestion
			`	"An obvious pursuit for the young that is," it nods, "if advice you I may, to not walk this way in a careless manner, the galaxy holds many dangers that your kind has yet to discover or even to understand." It straightens up to its full height, towering over you, "but in the end worth the trouble it will be."`
```

				goto "end"
			label "new species"

> 			`	"What a peculiar task that is," it looks confused and tilts its head, "what to achieve by this you hope? That an incredible amount of dangers come with meeting other species you not know? For a reason invisible borders in space there are. To be careful and consider every last step I advice."`

Reviewer 5: ```suggestion
			`	"What a peculiar task that is," it looks confused and tilts its head, "what to achieve by this you hope? That an incredible amount of dangers come with meeting other species you not know? For a reason invisible borders in space there are. To be careful and consider every last step I advise."`
```

			label "end"
			`	After a short pause it continues talking, "Your presence here our kind accepts, although not very happy we are. Allowed to travel here you are, but conditions there are. Not too far from here another kind just starts growing, great plans for them our leaders have. Do not interrupt them, hostile actions against them we will revenge."`
			`	With these words the Quarg turns around and leaves. It seems like you are free to use the local facilities now.`

# Maybe add a version of the ships sprite that looks damaged/modded.
ship "Nimbus" "Nimbus FTL"
	add attributes
		"bunks" -25
	outfits
		"Liquid-Class Shield"
		"Liquid Hydrogen Cell"
		"Heavy Machine Gun" 20

		"MH Red-Class Engine"
		"MH Red-Class Steering"

mission "Incipias: The derelict"
	landing
	invisible
	to offer
		has "Incipias: Meet the Quarg: offered"
	npc assist
		government "Hicemus"
		ship "Nimbus FTL" "HIC Veni #000001"
		system "Hui'uc"
		to despawn
			has "incipias: helped the derelict"
		on assist
			conversation

> 				`When you approach the ship you get hailed and as you accept the hail color gradients start moving over your screen but your speakers stay silent. It seems like communication over these channels is doomed to fail, you continue your boarding maneuver.`

Reviewer 5: This should be rephrased.

				scene "scene/incipias message"

> 				`	You manage to dock to the disabled ship quite easily. The lock opens and a hallway opens up before you, obviously not built for humans, the walls are strangely curved and the floor has nearly no grip at all. But your view gets caught by the five giant aliens half octopode, half jellyfish, they are pulsating in bright colors, similar to the gradients that appeared on your screen earlier. One of them is floating behind a giant mounted gun, the two barrels are pointed right at you. The gunner flashes in an orange light, and you can feel the tension in the room rising.`

Reviewer 5: ```suggestion
				`	You manage to dock to the disabled ship quite easily. The lock opens and a hallway opens up before you, obviously not built for humans. The walls are strangely curved and the floor has nearly no grip at all. Your view gets caught by the five giant aliens half octopode, half jellyfish that are pulsating in bright colors, similar to the gradients that appeared on your screen earlier. One of them is floating behind a giant mounted gun, the two barrels are pointed right at you. The gunner flashes in an orange light, and you can feel the tension in the room rising.`
```

				choice

> 					"Try to convey that you intend no harm."

Reviewer 5: ```suggestion
					`(Try to convey that you intend no harm.)`
```
These might also be missing indentation?

						goto peace

> 					"Drop the repair equipment and run back into your ship."

Reviewer 5: ```suggestion
					`(Drop the repair equipment and run back into your ship.)`
```

						goto flee

> 					"Try to make a non lethal shot at the one behind the turret and run!"

Reviewer 5: ```suggestion
					"Try to make a non-lethal shot at the one behind the turret and run!"
```

> 					"Try to make a non-lethal shot at the one behind the turret and run!"

Reviewer 5: ```suggestion
					`(Try to make a non-lethal shot at the one behind the turret and run!)`
```

				`	With the repair equipment delivered there is nothing else you can do and communication is not working so you turn around and leave. Just as you leave you can see how the aliens start pulsating in all colors of the spectrum, gradients are moving over their skin so fast that it almost looks white.`
					goto end
				label flee
				`	As the tension starts raising even further, you just drop the repair equipment and run back into your ship, you smash the button that closes the lock. You get a last look on the alien, blue and yellow tones are mixing on their skin, but they are not taking any action.`
				label end

> 				`	Apparently they figured how to use the repair equipment, as you fly away the engines of the ship are booting back up.`

Reviewer 5: ```suggestion
				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment.`
```

> 				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen, notably this ship had no Hyperdrive or anything similar installed, how strange considering there is no inhabitated planet in this system. Maybe you should talk to the Quarg and see if they can help.`

Reviewer 5: ```suggestion
				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen, notably this ship had no hyperdrive or anything similar installed, how strange considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`
```

> 				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen, notably this ship had no hyperdrive or anything similar installed, how strange considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`

Reviewer 5: ```suggestion
				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen. Notably this ship had no hyperdrive or anything similar installed; strange considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`
```

				action
					set "incipias: helped the derelict"
		personality
			confusion 50
			derelict uninterested staying
	on enter "Hui'uc"
		conversation
			`On entering the system you find an alien spaceship disabled and badly damaged. You remember your training as a captain when your teacher told you about something about a codex of captains, "A captain never leaves a ship in need behind."`

mission "Incipias: Help the stranded Part 1"
	landing
	source
		government "Quarg (Incipias)"
	destination "Pon'tes"
	to offer
		has "incipias: helped the derelict"
	on offer
		conversation

> 			`Right after you arrived on the planet you immedeatly go looking for a Quarg official to tell about the stranded ship. Luckily it does not take long before you find one.`

Reviewer 5: ```suggestion
			`Right after you arrived on the planet you immediately go looking for a Quarg official to tell about the stranded ship. Luckily it does not take long before you find one.`
```

> 			`Right after you arrived on the planet you immediately go looking for a Quarg official to tell about the stranded ship. Luckily it does not take long before you find one.`

Reviewer 5: ```suggestion
			`Right after you arrived on the planet you go looking for a Quarg official to tell about the stranded ship. Luckily it does not take long before you find one.`
```

			`	"Human, in quite a haste you seem to be. What the so pressing matter could be?" It asks as it clearly recognizes that you are a bit out of breath.`
			choice

> 				`	Explain the situation.`

Reviewer 5: ```suggestion
				`	(Explain the situation.)`
```

					goto explanation
				`	"I need advice regarding the species in this region."`
			`	It looks at you curiously, "So you met them? I am surprised that you were able to communicate, it took a very long time even for us to decipher their way of communication."`

> 			`	You quickly explain what happened and the Quarg seens very relieved about the fact that you were not able to talk to them, it probably was a big achievement for them.`

Reviewer 5: ```suggestion
			`	You quickly explain what happened and the Quarg seems very relieved about the fact that you were not able to talk to them, it probably was a big achievement for them.`
```

			label explanation
			`	"A most peculiar situation you ran into, should not be able to cross the borders of their system yet this species..." It tilts it head for a moment considering what you just said. "To help them I would love, but no ships we have that on gas giants can land and too early it is for us to reveal ourselves to them, I fear nothing there is we can do." It shakes its head.`
			choice
				`	"Then let me help them."`
					goto initiative

> 				`	"How can you make such a cold hearted decision?"`

Reviewer 5: ```suggestion
				`	"How can you make such a cold-hearted decision?"`
```

			`	"This difficult for you to understand might be, but the greater good over the sole life I have to place. If ourselves show to them now by the shock of such and advanced species existing will be created chaos."`
			choice
				`	"Then let me help them."`
					goto initiative
				`	"You can deal with the consequences later, help them now!"`
			`	"Insisting you sure are human, understand I can your point but intervention by us out of question it is. But a proposal for you I have, to you, a solo captain the means to help I will grant, of course some conditions there are."`
			choice
				`	"Sure let me do the work."`
				`	"If you are too weak..."`
			`	Something like a giggle comes from the Quarg, before continuing.`
			label initiative
			`	"Your initiative most welcome is human, but not the tools for such a mission you have. To my knowledge no vessels capable of landing on gas giants ever developed by your kind. And as said you, not capable of communicating with them you are."`

> 			`	"But help you I can at one point, communication that is. I can provide you with a single one of our translators, which will allow you to contact the species in their homesystem. To sell you one of their ships ask to them, easy it wont be."`

Reviewer 5: ```suggestion
			`	"But help you I can at one point, communication that is. I can provide you with a single one of our translators, which will allow you to contact the species in their home system. To sell you one of their ships ask to them, easy it won't be."`
```

			`	It pauses before getting very serious, "but remember one thing you have to, do not to give them any advanced technology of yours. If you agree to that point help you we shall."`
			choice
				`	"If that is how I can help them."`
					goto yes
				`	"Yes, but why do you want me not to sell my tech?"`
			`	"Interrupting the natural grow of a species not predictable the consequences are even letting you contact them a great risk is. But that one willing to take I am for the lives of these souls."`
			label yes

> 			`	"Happy I am that you agree to that condition. Please for a moment wait here," it says and hurrilly walks away.`

Reviewer 5: ```suggestion
			`	"Happy I am that you agree to that condition. Please for a moment wait here," it says and hurriedly walks away.`
```
Maybe `quickly` or `in a hurried manner` or leave it out altogether.

Author: Leaving it out seems like the best option

Author: ```suggestion
			`	"Happy I am that you agree to that condition. Please for a moment wait here," it says and walks away.`
```

			`	It only takes it a moment to get back carrying a sleek silver box with a display on one side.`

> 			`	"Our translator this is. Into your language it can translate thanks to our database. Be aware of the honor of me handing this you," it says reaching it to you.`

Reviewer 5: ```suggestion
			`	"Our translator this is. Into your language it can translate thanks to our database. Be aware of the honor of me handing this you," it says, handing the translator over.`
```

> 			`	As you stow it away it explains how to use it. "Oh and captain one last thing, the complexety of their language even we can not express, the translator might not be able to translate some parts directly, when it happens you will notice."`

Reviewer 5: ```suggestion
			`	As you stow it away it explains how to use it. "Oh and captain one last thing, the complexity of their language even we can not express, the translator might not be able to translate some parts directly, when it happens you will notice."`
```

			`	The quarg wishes you best of luck and leaves you alone. Time to head to Dom'us.`
				accept
	on enter Dom'us
		"reputation: Hicemus" += 10
		conversation
			`Entering the system with the translator, you prepare to broadcast your first message.`
			choice

> 				`	"Hello I am captain <first> <last> here to make contact."`

Reviewer 5: ```suggestion
				`	"Hello. I am captain <first> <last> here to make contact."`
```

> 				`	"Greetings strangers I seek to help you."`

Reviewer 5: ```suggestion
				`	"Greetings, strangers. I seek to help you."`
```

				# silly option will probably remove it but for now it is just funny
				`	"Howdy aliens, I come in peace.`

> 			`	For a minute, nothing happens at all, it seems like a communication went off.`

Reviewer 5: > it seems like a communication went off

What does this mean?

Author: communication is happening

> 			`	But then a motion goes through all ships in the system and then a channel opens from the space station. Colors rush over your display a mix of yellow and green, you point the translator on the display as the Quarg told you.`

Reviewer 5: ```suggestion
			`	After some time, a motion goes through all ships in the system and then a channel opens from the space station. Colors rush over your display a mix of yellow and green, and you point the translator to the display as the Quarg told you.`
```

			`	"Welcome [stranger, danger] we greet you in [this space, our home]."`
			`	The colors stop before rushing over your screen again.`
			`	"What is your [intention, purpose] here?"`
			choice
				`	"I am here to make peaceful contact with you."`
				`	"I want to help stranded ones of your kind."`
			`	Again a long pause. "Your intentions are [kind, noble]. Please dock to [our station, our capital]."`
			`	Along the message comes a small plan of the station with the docking ports highlighted.`
	on complete
		conversation

> 			`Thanks to the plans you are easily able to dock the station. Your lock opens and you enter an environment like nothing you ever saw before. There are no visible pathways but foggy hallways some even going up vertical. A crowd of these beings gathered in front of your lock each pulsating in a multitude of colors. A group of five floats in front of the crowd, the middle one starts communicating with you pulsating in calm colors.`

Reviewer 5: ```suggestion
			`Thanks to the plans you are easily able to dock with the station. Your airlock opens and you enter an environment like nothing you ever saw before. There are no visible pathways but foggy hallways some even going up vertically. A crowd of beings gathered in front of your airlock each pulsating in a multitude of colors. A group of five floats in front of the crowd. The middle one starts communicating with you pulsating in calm colors.`
```

			`	"Welcome [otherworldly being, standing one] we are the Incipias, welcome to [our home, our capital]."`
			`	It stops as if awaiting an answer.`
			choice

> 				`	"Greetings Incipias I am a human."`

Reviewer 5: ```suggestion
				`	"Greetings, Incipias. I am a human."`
```

				`	"I thank you for your hospitality."`
			`	A movement goes through the crowd colors now going through them like waves.`
			`	The speaker (or would you call him painter?) continues, "You are the first other being that ever [communicated, made friends]. Are you the only [danger, other being] out there?"`
			choice

> 				`	"No there are others too."`

Reviewer 5: ```suggestion
				`	"No, there are others too."`
```

				`	"The galaxy is full of other beings."`

> 			`	Another wave goes through the crowd. "So there are more. But, stranger how can we [help, befriend] you?"`

Reviewer 5: ```suggestion
			`	Another wave goes through the crowd. "So there are more. But, stranger, how can we [help, befriend] you?"`
```

			choice

> 				`	Tell them about the stranded and how you want to help them.`

Reviewer 5: ```suggestion
				`	(Tell them about the stranded and how you want to help them.)`
```

					goto stranded
				`	"I want one of your ships."`
			`	"[I, we] shall allow you to buy one. If that is all that it takes to make you a [friend, not danger]. If that is all please excuse us, we need to talk about what this new situation means for us, I hope we can [talk to, question] you more another time."`
				goto end
			label stranded
			`	Another wave goes through the crowd but this time you notice an overflow in green tones.`
			`	"So they [are still alive, made the big step]. We shall give you access to our [ships, achievements]. Let us prepare the shipyard, we hope to [speak to, question] you more on another occasion.`
			label end

> 			`	The five float away but the crowd stays looking at you, probably out of interest.`

Reviewer 5: ```suggestion
			`	The five float away but the crowd stays continues staring at you, probably out of interest.`
```

			`	You should get yourself a ship and install a hyperdrive on it to save the stranded ones in the Hui'uc system.`

mission "Incipias: Help the stranded Part 2"
	landing
	passengers 5
	source
		planet Turra
	destination "Pon'tes"
	to offer
		has "Incipias: Help the stranded Part 1: done"
	on offer
		conversation

> 			`Your ship dives into the clouds of the gas giant an environment of a scale you never experienced before giant clouds tower like castles. Sometimes the clouds open to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`

Reviewer 5: ```suggestion
			`Your ship dives into the clouds of the gas giant, an environment on a scale you never experienced before. Giant clouds tower like castles, sometimes opening to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`
```

> 			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen. You maneuver your ship to that point. The stranded ship looks even worse than the first time the storms seem to have damaged some of the hull. The shields must have broken when they ran out of fuel.`

Reviewer 5: ```suggestion
			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen. You maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull. The shields must have broken when they ran out of fuel.`
```

> 			`	As you hail them you receive no answer, your scanners indicate that it ran out of energy.`

Reviewer 5: ```suggestion
			`	You receive no answer to your hail. Your scanners indicate that it ran out of energy.`
```

> 			`	Through a complex maneuver you are able to dock their ship.`

Reviewer 5: ```suggestion
			`	You carefully maneuver your ship closer, until you can dock with the Incipias ship.`
```

> 			`	When the lock opens five Incipias float in front of you, as they notice you they quickly back of, they were not expecting a human in one of their kind's ships. But then they seem to recognize you.`

Reviewer 5: ```suggestion
			`	When the lock opens five Incipias float in front of you. Noticing your presence they quickly back off, they were clearly not expecting a human in one of their kind's ships. It takes a moment for them to recognize you.`
```

			choice
				`	"Hello, I have come to save you."`

> 				`	"I have been send by your people to get you home."`

Reviewer 5: ```suggestion
				`	"I have been sent by your people to get you home."`
```

			`	They seem to switch from a tense state to a surprised one.`
			`	One of them starts pulsating, "You [speak, light] our language? How [surprising, good]. We would not have survived [much longer, a single day] here. You are our savior. We would love to talk more, but a [storm, calamity] is coming, we should leave now."`
			`	You tell them to come aboard and launch away from the planet.`
				accept
	on complete
		conversation

> 			`When you land on the station your passengers thank you again and leave, probably to get some well deemed rest.`

Reviewer 5: ```suggestion
			`When you land on the station your passengers thank you again and leave, probably to get some well-earned rest.`
```

> 			`	As you step out of the an Incipias approaches you, "Follow me [stranger, helper] before a crowd gathers, we would like to talk."`

Reviewer 5: ```suggestion
			`	As you step out of the <ship> an Incipias approaches you, "Follow me [stranger, helper] before a crowd gathers, we would like to talk."`
```

> 			`	As you step out of the <ship> an Incipias approaches you, "Follow me [stranger, helper] before a crowd gathers, we would like to talk."`

Reviewer 5: ```suggestion
			`	As you step out of the <ship> an Incipias approaches you. "Follow me [stranger, helper] before a crowd gathers, we would like to talk."`
```

> 			`	It guides you through the hallways avoiding larger groups, luckily it only chooses horizontal corridors. You arrive in a large room with a view over the entire station. Three Incipias are already in the room. One starts glowing.`

Reviewer 5: ```suggestion
			`	It guides you through the hallways avoiding larger groups, only moving through horizontal corridors. You arrive in a large room with a view over the entire station. Three Incipias are already in the room. One starts glowing.`
```

> 			`	"Welcome, we are the leaders of the Hicemus the [reigning, most powerfu] government of the Incipias. If you agree we would like to ask you a few questions. In return we would be open for questions too."`

Reviewer 5: ```suggestion
			`	"Welcome, we are the leaders of the Hicemus the [reigning, most powerful] government of the Incipias. If you agree we would like to ask you a few questions. In return we would be open for questions too."`
```

> 			`	"Welcome, we are the leaders of the Hicemus the [reigning, most powerful] government of the Incipias. If you agree we would like to ask you a few questions. In return we would be open for questions too."`

Reviewer 5: ```suggestion
			`	"Welcome. We are the leaders of the Hicemus, the [reigning, most powerful] government of the Incipias. If you agree we would like to ask you a few questions. In return we would be open for questions too."`
```

			choice
				`	"Sorry but I have other stuff to do."`
					goto endNoTime
				`	"Sure, ask something."`
			`	A green glow comes from all of them, "Thank you for taking the time. Can you tell us more about the [unknown territory, galaxy] surrounding us?"`
			choice

> 				`	Give them an exhausting story of what you have seen so far.`

Reviewer 5: ```suggestion
				`	(Give them an exhausting story of what you have seen so far.)`
```

					goto detail

> 				`	Give a rough overview of what you have seen.`

Reviewer 5: ```suggestion
				`	(Give a rough overview of what you have seen.)`
```

			`	After watching your explanation a quickfire of colors that your translator can not follow is exchanged between them, you mostly see yellow tones mixed with green and red.`

> 			`	"Thank you for this insight, all we [knew, gathered] until now was by observing stars to think that there is so much out there, it truly invokes determination to explore.`

Reviewer 5: ```suggestion
			`	"Thank you for this insight. All we [knew, gathered] until now was by observing stars. To think that there is so much out there, it truly invokes determination to explore.`
```

				goto yours
			label detail
			`	After watching your story a quickfire of colors that your translator can not follow is exchanged between them, you mostly see yellow tones mixed with green.`

> 			`	"Thank you for this exhaustive insight, all we [knew, gathered] until now was by observing stars close to us. To think that there is so much out there, it truly invokes determination to explore.`

Reviewer 5: ```suggestion
			`	"Thank you for this exhaustive insight. All we [knew, gathered] until now was by observing stars close to us. To think that there is so much out there, it truly invokes determination to explore.`
```

			label yours
			`	"But now to you, is there anything you would like to know?"`
			# might add to this list over time
			label choice
			choice

> 				`	"I do not have anymore questions."`

Reviewer 5: ```suggestion
				`	"I do not have any more questions."`
```

					goto end
				`	"Why do you call this station your capital?"`
					goto capital
				`	"Why can I not land on your home planet?"`

> 			`	A flash of red appears for only a second, "Our [old home, origin] is not directly ruled by us, they may approach you separately at some point in the future, be [careful, disgusted] of their lights. They have outdated values long laid of by us, who are progressing fast.`

Reviewer 5: ```suggestion
			`	A flash of red appears for only a second, "Our [old home, origin] is not directly ruled by us. They may approach you separately at some point in the future, be [careful, disgusted] of their lights. They have outdated values long laid off by us, who are progressing fast.`
```

				goto choice
			label capital

> 			`	The one pulsating now takes on a bright green color, "This station is our greatest achievement it is our [gate way, bridge] to the stars. In honor of our urge to [expand, explore] we made this station our capital to demonstrate the determination."`

Reviewer 5: ```suggestion
			`	The one pulsating now takes on a bright green color, "This station is our greatest achievement it is our [gateway, bridge] to the stars. In honor of our urge to [expand, explore] we made this station our capital to demonstrate the determination."`
```

> 			`	The one pulsating now takes on a bright green color, "This station is our greatest achievement it is our [gateway, bridge] to the stars. In honor of our urge to [expand, explore] we made this station our capital to demonstrate the determination."`

Reviewer 5: ```suggestion
			`	The one pulsating now takes on a bright green color, "This station is our greatest achievement, our [gateway, bridge] to the stars. In honor of our urge to [expand, explore] we made this station our capital to demonstrate our determination."`
```

				goto choice
			label end
			`	"Thank you again for your time, you gave us a lot to [process, store away]. We hope you too had a benefit from this."`
			`	You say goodbye and as you leave the room you can see a rush of colors reflecting on the walls.`
				decline
			label endNoTime

> 			`	"[I, we] [understand, are confused], safe travels captain." the representative says, you can notice a slight blue color in his communication.`

Reviewer 5: ```suggestion
			`	"[I, we] [understand, are confused]. Safe travels, captain." You notice a slight blue tone in the representative's communication.`
```

# Copyright (c) 2023 by RisingLeaf (https://github.com/RisingLeaf)
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Incipias: Guardian of Incipias Space"
	invisible
	landing
	repeat
	to offer
		not "Incipias: Guardian of Incipias Space: active"
	to complete
		never
	npc save
		government "Drak (Incipias)"
		personality heroic vindictive frugal forbearing
		system Il'le

> 		ship "Archon (Hyperdrive Only)" "Guardian of Spring"

Reviewer 2: ```suggestion
		ship "Archon (Hyperdrive Cloaked)" "Guardian of Spring"
```

			choice
				`	"Why do you care about these 'flowers'?"`
					goto care
				`	"What I search for is not technology."`
			`	The voice now takes on a calm tone. "Your persistence on this topic is admirable, but allow me to explain my conviction with an example."`
			label care
			`	"Why do humans care about art?" The voice now seems far away yet more present than ever. "Because it amazes you, the same way it amazes us to see another kind grow and overcome their burdens. The lives of species are like a painting that never stop being painted, but when they do stop it is a tragic thing to watch. That is why we try to protect these creations of life - so we can see what heights they reach."`
			`	There is a pause. "I see that you can not be convinced by me. Even though I could simply wipe you out here and now, I do not favor violence." The voice is now resolute in tone. "I will let you pass, but I will be watching you, as I am the guardian of this space."`
			`	The voice fades out leaving nothing behind but the feeling of being watched.`
				decline
			label agree
			action
				set "Incipias: accepted the guardians warning"

> 			`	"You are understanding, human, so I thank you in the name of this region." The voice fades away, leaving only the feeling of being watched."`

Author: Hm I wonder if it is alright to name drop them here already

> 			`	"You are understanding, human, so I thank you in the name of this region." The voice fades away, leaving only the feeling of being watched."`

Reviewer: I think it's fine. It adds some mystery and intrigue for the player, kind of like how the player can hear mentions of the Korath in human and Hai space before fully knowing what a Korath is.

				goto end
			label stranded
			`	Another wave goes through the crowd, but this time you notice an overflow in green tones.`
			`	"So they are still alive. We shall give you access to our ship. Let us prepare the ship-seller. Do note though that they can not jump between system-places, not like your own ship. We hope to question you more on another future-time.`
			label end
			`	The five float away but the crowd continues staring at you, probably out of interest.`

mission "Incipias: No help for the stranded"
	landing
	invisible
	source
		planet Turra
	destination "Pon'tes"
	to offer
		or
			has "Incipias: Help the stranded Part 1: declined"
		not "Incipias: Help the stranded Part 1: offered"
	on offer
		conversation
			`You see the stranded ship on your scanners, but with no way to communicate you have no way to help them. You have no choice but to leave them to their doom on this stormy world.`

mission "Incipias: Help the stranded Part 2"
	landing
	passengers 5
	source
		planet Turra
	destination "Pon'tes"
	to offer
		has "Incipias: Help the stranded Part 1: done"
	on offer
		conversation

> 			`Your ship dives into the clouds of the gas giant, an environment on a scale you never experienced before. Giant clouds tower like castles, sometimes opening to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`

Reviewer: ```suggestion
			`Your ship dives into the clouds of the gas giant, entering an environment on a scale you never experienced before. Giant clouds tower like castles, sometimes opening to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`
```

			choice
				`	"Why do you care about these 'flowers'?"`
					goto care
				`	"What I search for is not technology."`
			`	The voice now takes on a calm tone. "Your persistence on this topic is admirable, but allow me to explain my conviction with an example."`
			label care
			`	"Why do humans care about art?" The voice now seems far away yet more present than ever. "Because it amazes you, the same way it amazes us to see another kind grow and overcome their burdens. The lives of species are like a painting that never stop being painted, but when they do stop it is a tragic thing to watch. That is why we try to protect these creations of life - so we can see what heights they reach."`
			`	The voice pauses before continuing. "I see that you can not be convinced by me. Even though I could simply wipe you out here and now, I do not favor violence. I will let you pass, but I will be watching you as I am-" you can hear the ultimate resolution in the voice, "-the guardian of this space."`
			`	The voice fades out leaving nothing behind but the feeling of being watched.`
				decline
			label agree
			action
				set "Incipias: accepted the guardians warning"

> 			`	"You are understanding human, I thank you in the name of this region." The voice fades away lwaving only the feeling of being watched behind."`

Reviewer 2: ```suggestion
			`	"You are understanding, human, so I thank you in the name of this region." The voice fades away leaving only the feeling of being watched behind."`
```

				decline

mission "Incipias: Presence of the Guardian"
	invisible
	landing
	to complete
		never
	on enter Dom'us
		conversation
			`As you enter the system you still have a feeling of being watched, the presence from the previous system still ligering around you.`
				to display
					not "Incipias: accepted the guardians warning"

> 			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on atempts to decieve me, human. Know that you will be watched." As the voice fades away a strong presence stays around you.`

Reviewer 2: ```suggestion
			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." As the voice fades away a strong presence stays around you.`
```

				to display
					has "Incipias: accepted the guardians warning"
				decline
	on enter Hui'uc
		conversation
			`As you enter the system you still have a feeling of being watched, the presence from the previous system still ligering around you.`
				to display
					not "Incipias: accepted the guardians warning"

> 			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on atempts to decieve me, human. Know that you will be watched." As the voice fades away a strong presence stays around you.`

Reviewer 2: ```suggestion
			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." As the voice fades away a strong presence stays around you.`
```

				goto "both planets"
			label "spec inci"
			`	You have to wait for a few hours until a group of three Quarg appear. The one in the middle starts talking in a very strange accent.`
			label "both planets"
			`	"Welcome human. Your lengthy travels have brought you here, yet your kind has never set foot in this region of space. May we inquire as to your mission?"`
			choice
				`	"I am just exploring the galaxy."`
					goto "explorer"
				`	"I am looking for new species."`
					goto "new species"
			label "explorer"
			`	"It is an obvious pursuit for the young," it nods. "If I may advise you, do not walk this way in a careless manner. The galaxy holds many dangers that your kind has yet to discover or even to understand." It straightens up to its full height, towering over you.`
				goto "end"
			label "new species"
			`	"What a peculiar task that is." It looks confused and tilts its head. "What do you hope to achieve by this? That an incredible amount of dangers come with meeting other species, did you not foresee? The boundaries of space are not idle accidents. I advise you to be careful and consider every last step."`
			label "end"

> 			`	After a short pause it continues talking; you notice an almost-hidden frustration lurking beneath the Quarg's face. "You are permitted to travel here, but there are conditions. Not too far from here another kind has begun their journey to the stars. Do not interrupt them for your own sake, as we will avenge any hostile actions against them."`

Author: there is no ringworld

				label end
				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment. Instead of leaving the system, however, the ship quietly disappears into the nearest gas giant.`
				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen. Notably this ship had no hyperdrive or anything similar installed; strange considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`
				action
					set "incipias: helped the derelict"
		personality
			confusion 50
			derelict uninterested staying
	on enter "Hui'uc"
		conversation
			`On entering the system you find an alien spaceship disabled and badly damaged. There do not appear to be any other ships or inhabited planets in the system.`

mission "Incipias: Help the stranded Part 1"
	landing
	source
		government "Quarg (Incipias)"
	destination "Pon'tes"
	to offer
		has "incipias: helped the derelict"
	on offer
		conversation

> 			`Right after you arrive on the planet, you search for a Quarg who speaks your language so that you can tell them about the stranded ship you found. One of the Quarg seems to understand your intention and gestures for you to wait. It returns a few hours later accompanied by the same Quarg you talked to before.`

Author: Added a small question.

			`	"Human, you seem to be in quite a haste. What could be such a pressing matter?" it asks.`
			choice
				`	(Explain the situation.)`
					goto explanation
				`	"I need advice regarding the species in this region."`
			`	It looks at you curiously. "So you met them? I am surprised that you were able to communicate. It took a very long time even for us to decipher their way of communication."`
			`	You quickly explain what happened. The Quarg seems very relieved about the fact that you were not able to talk to them.`
			label explanation
			`	"You ran into a most peculiar situation. This species should not be able to cross the borders of their system yet." It tilts it head for a moment considering what you just said. "It is our desire to help them, but it is too early for us to reveal ourselves to their species. I fear there is nothing we can do." It shakes its head.`
			choice
				`	"Then let me help them."`
					goto initiative
				`	"I see, there is nothing to do."`
					goto declined
				`	"How can you make such a cold-hearted decision?"`

> 			`	"This might be difficult for you to understand, but we must place the greater good above individual lives. If we show ourselves to them now, the shock of such an advanced species appearing in their space will fan the flames of chaos."`

Reviewer: ```suggestion
			`	"This is perhaps difficult to understand for an adolescent species such as yourself, but we must place the continuation of species above the existence of individuals. If we reveal ourselves to them now, the shock of such an advanced species appearing in their space will fan the flames of chaos."`
```

			choice
				`	"Then let me help them."`
					goto initiative
				`	"You can deal with the consequences later. Help them now!"`
			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can foresee the likelihood of you making a rash and impetuous decision to contact this ship."`
			choice
				`	"Sure, let me do the work."`
				`	"If you are too weak..."`

> 			`	The Quarg rappidly clicks it teeths, before continuing.`

Reviewer: ```suggestion
			`	The Quarg rapidly clicks its teeth, before continuing.`
```

			label initiative
			`	"You lack the tools for such a mission. To my knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense."`
			`	"But for the sake of the Incipias, I will assist you on this last point. If you cannot be dissuaded from your course of action, then it is best you are given a means of communication, so that your interactions can at least be on amicable terms. I can provide you with a single one of our translators, which will allow you to contact the species in their home system. Request that they sell you one of their ships, so that you are able to land on their planets."`
			`	It pauses before taking on a more serious tone. "However, you must remember one thing. You are forbidden from granting them any advanced technology of yours. If you agree to that point we shall assist you."`
			label choice
			choice
				`	"If that's how I can help them, sure."`
					goto yes

> 				`	"Yes, but why do you not want me to sell my tech?"`

Reviewer: ```suggestion
				`	"Why do you not want me to sell my tech?"`
```

			`	"Halting the natural increase of a species is unpredictable at the best of times. Your species lacks the wisdom to understand this, as you are young, and have not seen what we have seen." The Quarg pauses for a moment, lost in thought. "The foreseeable risks of even letting you contact them are monumental, and not something you should consider trivial."`
				goto choice
			label yes
			`	"Happy I am that you agree to that condition. Bear in mind, Captain, that the slightest misstep may have grave consequences." The Quarg stares you down. "We will be watching your actions, to make sure that they are peaceful and without ill or unwise intent. Please for a moment wait here," it says, and walks away.`
			`	The Quarg only takes a moment to come back; it is now carrying a sleek silver box with a display on one side.`

> 			`	"This is our translator, our link to the Incipias. It is able to translate into your language as well. Be aware of the honor of me handing this to you," it says.`

Reviewer: ```suggestion
			`	"This is our translator, our link to the Incipias. It is able to translate their unwonted language into your speech as well. Be aware of the honor of me gifting this to you," it says.`
```

> 			`	As you stow it away, the Quarg speaks once more. "To satisfactorily utilize this device, you must point it towards the source of the communication, so that it is able to pick up on the emission. Given the complexity of their language, the translator might not be able to adequately translate some parts of their speech. You will understand it when it happens."`

Reviewer: ```suggestion
			`	As you stow it away, the Quarg speaks once more. "To satisfactorily utilize this device, you must direct it towards the source of the communication, so that it is made able to discern the relevant emissions. Given the complexity of their language, the translator might not be able to adequately translate some aspects of their obscure speech."`
```

			`	It then looks you directly in the eyes. "Remember what I have told you, human. The stars have no place for those who would act foolishly and thoughtlessly towards their fellow beings." The Quarg then walks away, leaving you alone.`
				accept
			label declined

> 			`	"Thank you for your understanding human, have safe travels." With these words the Quarg turns away.`

Reviewer: One last reminder to the player to not attack Incipias ships should they decide to visit the Incipias' home system.
```suggestion
			`	"We are gracious for your understanding, but remember thus: the Incipias are under our watch and care. None shall come to harm them without incurring our wrath." With these words the Quarg turns away.`
```

Author: ah nice one

			label end
			`	The five float away but the crowd continues staring at you, probably out of interest.`

mission "Incipias: No help for the stranded"
	landing
	invisible
	source
		planet Turra
	destination "Pon'tes"
	to offer
		or
			has "Incipias: Help the stranded Part 1: declined"
		not "Incipias: Help the stranded Part 1: offered"
	on offer
		conversation
			`You see the stranded ship on your scanners but with no way to communicate you have no way to help them and have no choice but to leave them to their doom on this stormy world.`

mission "Incipias: Help the stranded Part 2"
	landing
	passengers 5
	source
		planet Turra
	destination "Pon'tes"
	to offer
		has "Incipias: Help the stranded Part 1: done"
	on offer
		conversation
			`Your ship dives into the clouds of the gas giant, an environment on a scale you never experienced before. Giant clouds tower like castles, sometimes opening to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`

> 			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and the shield generator must have broke when they ran out of fuel, because their shields are completely down.`

Author: not in game mechanics, but lore wise they use some sort of fuel, however it wont run out for the duration of a flight

> 			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and the shield generator must have broke when they ran out of fuel, because their shields are completely down.`

Reviewer: I think you can add a tiny amount of fuel consumption to Incipias shield outfits then.

> 			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and the shield generator must have broke when they ran out of fuel, because their shields are completely down.`

Author: sure makes them more interesting

> 			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and the shield generator must have broke when they ran out of fuel, because their shields are completely down.`

Reviewer: ```suggestion
			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and their shields are completely down - the shield generator must have gone offline when they ran out of fuel.`
```

			`	You receive no answer to your hail. Your scanners indicate that it ran out of energy.`
			`	You carefully maneuver your ship closer, until you can dock with the Incipias ship.`
			`	When the lock opens five Incipias float in front of you. Noticing your presence they quickly back off; they were clearly not expecting a human in one of their kind's ships. It takes a moment for them to recognize you.`
			choice
				`	"Hello, I have come to save you."`
				`	"I have been sent by your people to get you home."`
			`	They seem to switch from a tense state to a surprised one.`
			`	One of them starts pulsating, "You light our language? How surprising. We would not have survived another day here. You are our life-protector. We would love to talk more, but a weather-calamity is coming. We should leave now."`
			`	You tell them to come aboard, and you manage to launch your ship just before a storm hits the area.`
				accept
	on complete
		conversation
			`When you land on the station your passengers thank you again and leave, probably to get some well-earned rest.`

> 			`	As you step out of the <ship> an Incipias approaches you. "Follow me outer-helper, before a crowd gathers. We would like to talk."`

Author: some more lore (also next commit)

			`	A green glow comes from all of them. "Thank you for taking the time-patience. Can you tell us more about the deep space surrounding us?"`
			choice
				`	(Give them an exhausting story of what you have seen so far.)`
					goto detail
				`	(Give a rough overview of what you have seen.)`
			`	After seeing your explanation a rapid-fire sequence of colors are exchanged between them. Your translator cannot follow, so you are left watching them flash in yellow tones mixed with green and red.`
			`	"Thank you for this insight. All we gathered until now was by observing-studying stars. To think that there is so much out there... it truly invokes a determination to explore.`
				goto yours
			label detail
			`	After watching your explanation a rapid-fire sequence of colors are exchanged between them. Your translator cannot follow, so you are left watching them flash in yellow tones mixed with green.`
			`	"Thank you for this exhaustive insight. All we gathered until now was by observing-studying stars close to us. To think that there is so much out there... it truly invokes a determination to explore.`
			label yours
			`	"But now to you. Is there anything you would like to know?"`

> 			# might add to this list over time

Author: added one more question, not sure what else to add, maybe some history lessons?
(yes, next commit)

			label choice
			choice
				`	"I do not have any more questions."`
					goto end
				`	"Why do you call this station your capital?"`
					goto capital
				`	"Why can I not land on your home planet?"`

> 			`	A flash of red appears for only a second. "Our origin-world is not directly ruled by us. They may approach you separately at some point in the future, but be careful of their lights. They have outdated values long laid off by us, who are progressing fast.`

Reviewer: ```suggestion
			`	A flash of red appears for only a second. "Our origin-world is not directly ruled by us. They may approach you separately at a future-point, but be careful of their lights. They have aged-time values long left-alone by us, who are progressing fast.`
```

				goto choice
			label capital

> 			`	The one pulsating now takes on a bright green color, "This station is our greatest achievement, our star-bridge. In honor of our urge to expand we made this station our capital to demonstrate our determination."`

Reviewer: ```suggestion
			`	The one pulsating now takes on a bright green color, "This station is our greatest action-pride, our star-bridge. In honor of our urge to expand we made this station our capital to demonstrate our determination."`
```

				goto end
			label "new species"
			`	"What a peculiar task that is." It looks confused and tilts its head. "What do you hope to achieve by this? That an unfathomable amount of dangers come with encountering and treating with other species, did you not foresee? The boundaries of space are not idle accidents. I advise you to be scrupulous and ponder every last step."`
			label end
			`	After a short pause, you notice an almost-hidden frustration lurking beneath the Quarg's face before it continues speaking. "You are authorized to continue your nomadic voyagings here, but there are demands we Quarg make of you. Within a trifling interval of this moon the neighboring kind has initiated their journey to the stars. Do not interrupt them for yours, or sake of their own."`
			`	With these words the Quarg turn around and leave. It seems like you are free to use the local facilities now.`

> # TODO: Maybe add a version of the ship's sprite that looks damaged/modded.

Reviewer 6: ```suggestion
```

			choice
				`	"No, there are others too."`
				`	"The galaxy is full of other beings."`
			`	Another wave goes through the crowd. "So there are more. But, far-stranger, how can we help you?"`
			choice
				`	(Tell them about the stranded ship and how I want to help them.)`
					goto stranded
				`	"I want one of your ships."`
			`	"We shall allow you to buy one, if that is all that it takes to make you a not-danger. Please excuse us. We need to talk about what this new situation-opportunity means for our species. I hope we can out-question you more another time."`
				goto end
			label stranded
			`	Another wave goes through the crowd, but this time you notice an overflow in green tones.`
			`	"So they are still alive. We shall give you access to our ships, so that you may enter the air-planet they have taken refuge in. Do note though that they cannot jump between system-places, not like your own ship. We hope to question you more on another future-time.`
			label end
			`	The five float away but the crowd continues staring at you, probably out of interest.`

> mission "Incipias: No Help For The Stranded"
> 	landing
> 	invisible
> 	source
> 		planet Turra
> 	destination "Pon'tes"
> 	to offer
> 		or
> 			has "Incipias: Help The Stranded 1: declined"
> 		not "Incipias: Help The Stranded 1: offered"
> 	on offer
> 		conversation
> 			`You identify the stranded ship on your scanners, but with no way to communicate with them you are unable to to help. You have no choice but to leave them to their doom on this stormy world.`

Reviewer 6: ```suggestion
mission "Incipias: No Help For The Stranded"
	landing
	invisible
	source
		planet Turra
	to offer
		has "Incipias: The Derelict: done"
		or
			has "Incipias: Help The Stranded 1: declined"
			not "Incipias: Help The Stranded 1: offered"
	on offer
		fail
		dialog
			`You identify the stranded alien ship you assisted earlier on your scanners, but with no way to communicate with them you are unable to to help. You have no choice but to leave them to their doom on this stormy world.`
```

			choice
				`	"Why do you care about these 'flowers'?"`
					goto care
				`	"What I search for is not technology."`
			`	The voice now takes on a calm tone. "Your persistence on this topic is admirable, but allow me to explain my conviction with an example."`
			label care
			action
				event "label incipias space"
			`	"Why do humans care about art?" The voice now seems far away yet more present than ever. "Because it amazes you, the same way it amazes us to see another kind grow and overcome their burdens. The lives of others are like a painting that never stops being painted, but is a tragic thing to watch the painting stop. That is why we try to protect these creations of life - so we can see what heights they reach."`
			`	There is a pause. "I see that you can not be convinced by me. Though I could intervene in your journey, I do not favor unsavory confrontation." The voice is now resolute in tone. "I will let you pass, but I will be watching you, as I am the guardian of this space."`
			`	The voice fades out leaving nothing behind but the feeling of being watched.`
				decline
			label agree
			action

> 				set "Incipias: accepted the guardians warning"

Author: changed

				event "label incipias space"
			`	"You are understanding, human, so I thank you in the name of the Incipias." The voice fades away, leaving only the feeling of being watched."`
				decline

mission "Incipias: Presence of the Guardian"
	invisible
	landing
	to complete
		never
	on enter Dom'us
		fail

> 		conversation

Author: done

			choice
				`	"Sure, ask me something."`
				`	"Sorry, but I have other stuff to do."`
					goto "end no time"
			`	A green glow appears on all of them. "Thank you for taking the time-patience to speak with us. Can you tell us more about the deep space-void surrounding us?"`
			choice
				`	(Give them an exhaustive story of what you have seen so far.)`
					goto detail
				`	(Give a rough overview of what you have seen.)`
			`	After "seeing" your explanation, a rapid-fire sequence of colors is exchanged between the Incipias. Your translator cannot follow, so you are left watching them flash in what end up being yellow tones mixed with green and red.`
			`	"Thank you for this insight. All we gathered until now was by observing-studying stars. To think that there is so much out there... it truly raise-strengthens a determination to explore.`
				goto yours
			label detail
			`	After "watching" your explanation, a rapid-fire sequence of colors is exchanged between the Incipias. Your translator cannot follow, so you are left watching them flash in what ends up being yellow tones mixed with green.`

> 			`	"Thank you for this profound-depth insight. All we gathered until now was by observing-studying stars close to us. To think that there is so much out there... it truly invokes a determination to explore.`

Reviewer 5: ```suggestion
			`	"Thank you for this profound-depth insight. All we gathered until now was by observing-studying stars close to us. To think that there is so much out there... it truly invokes a determination-will to explore.`
```

# Copyright (c) 2023 by RisingLeaf (https://github.com/RisingLeaf)
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Incipias: Warning of the Guardian"
	invisible
	landing
	to complete
		never
	on enter Il'le
		conversation

> 			`A voice begins to speak from somewhere. "Your path brings you to a dead end, traveler. Nothing for you lies further," a voice with an uncertain origin says.`

Reviewer 6: This reads like the first line slightly adjusted and put at the end too. Either I'd differentiate it from the first line enough (and avoid voice), or keep one of the two.

> 			`A voice begins to speak from somewhere. "Your path brings you to a dead end, traveler. Nothing for you lies further," a voice with an uncertain origin says.`

Reviewer: I'm not sure what you mean here? I took out a line that essentially duplicated the first part of the sentence.

> 			`A voice begins to speak from somewhere. "Your path brings you to a dead end, traveler. Nothing for you lies further," a voice with an uncertain origin says.`

Reviewer 6: I'm going crazy.

				decline
			label agree
			action
				set "Incipias: accepted the guardians warning"
				event "label incipias space"
			`	"You are understanding, human, so I thank you in the name of the Incipias." The voice fades away, leaving only the feeling of being watched."`
				decline

mission "Incipias: Presence of the Guardian"
	invisible
	landing
	to complete
		never
	on enter Dom'us
		fail
		conversation
			`The presence from the previous system somehow lingers as you continue to feel as though you're being watched.`
				to display
					not "Incipias: accepted the guardians warning"
			`As you enter the system, the same voice from before echoes through your mind, though this time, there seems to be a heavy weight behind it. "I do not take kindly to your deception, human. Know that you will be watched." The voice fades away, but the strong presence of something remains.`
				to display
					has "Incipias: accepted the guardians warning"
				decline
	on enter Hui'uc
		fail
		conversation
			`The presence from the previous system somehow lingers as you continue to feel as though you're being watched.`
				to display
					not "Incipias: accepted the guardians warning"

> 			`As you enter the system, the same voice from before echoes through your mind, though this time, there seems to be a heavy weight behind it. "I do not take kindly to your deception, human. Know that you will be watched." The voice fades away, but the strong presence of something remains.`

Author: wait are you removing the commas saug added? gonna have to look into that...

			label end
			`	After a short pause, you notice an almost-hidden frustration lurking beneath the Quarg's face before it continues speaking. "You are authorized to continue your nomadic voyagings here, but there are demands we Quarg make of you. Within a trifling interval of this moon the neighboring kind has initiated their journey to the stars. Do not interrupt them for yours, or sake of their own."`
			`	With these words the Quarg turn around and leave. It seems like you are free to use the local facilities now.`

# TODO: Maybe add a version of the ship's sprite that looks damaged/modded.
ship "Nimbus" "Nimbus FTL"
	add attributes
		"bunks" -25
	outfits
		"Liquid-Class Shield"
		"Metallic Hydrogen Cell"
		"Cloud Piercer" 20

		"MH Red-Class Engine"
		"MH Red-Class Steering"

mission "Incipias: The Derelict"
	landing
	invisible
	to offer
		has "Visit Quarg in Incipias Space: offered"
	npc assist
		government "Hicemus"
		ship "Nimbus FTL" "HIC Veni #000001"
		system "Hui'uc"
		to despawn
			has "incipias: helped the derelict"
		on assist
			conversation
				scene "scene/incipias message"

> 				`As you approach the derelict ship, it sends you a hail request. Once you accept the hail, color gradients suddenly start moving over your screen in a dizzying fashion, while curiously, there is no audio. It's clear that unless you have some sort of translation device, you will not be able to understand whatever transmission you just witnessed. You continue your attempt to assist the ship, preparing to bring some repair supplies on board.`

Reviewer: ```suggestion
				`As you approach the derelict ship, it sends you a hail request. Once you accept the hail, color gradients suddenly start moving over your screen in a dizzying fashion, but curiously there is no audio. It's clear that unless you have some sort of translation device, you will not be able to understand whatever transmission you just witnessed. You continue your attempt to assist the ship, preparing to bring some repair supplies on board.`
```

			`	"Thank you for this profound-depth insight. All we gathered until now was by observing-studying stars close to us. To think that there is so much out there... it truly invokes a determination to explore.`
			label yours
			`	"But now to you. Is there anything you would like to know?"`
			label choice
			choice
				`	"Why can I not land on your home planet?"`
				`	"Why do you call this station your capital?"`
					goto capital
				`	"How is life on a gas giant?"`
					goto oddquestion
				`	"I do not have any more questions."`
					goto end
			`	A flash of red appears for only a second. "Our origin-world is not directly ruled by us. They may approach you separately at a future-point, but be careful of their lights. They have aged-time values long left-alone by us, who are progressing fast.`
				goto choice
			label capital
			`	The one pulsating now takes on a bright green color, "This station is our greatest action-pride, our star-bridge. In honor of our urge to expand we made this station our capital to demonstrate our determination."`
				goto choice
			label oddquestion

> 			`	"What an odd-reasonable question." It's glow turns into a swirl of purple and green. "We could ask the same in return, what is it like to live on hard rock? But we understand its reasoning-origin. So let me tell you of the cloud-halls of our home-sky. Lightning will draw the most beautiful color-sings between the pillar-clouds between them are our cities, the most color-lifeful places you will ever see. This is a bit poetic-gradient description but I do not know what else to say."`

Reviewer: ```suggestion
			`	"What an odd-reasonable question." It's glow turns into a swirl of purple and green. "We could ask the same in return - what is it like to live on hard rock? But we understand its reasoning-origin. So let me tell you of the cloud-halls of our home-sky. Lightning will draw the most beautiful color-sings between the pillar-clouds. Between them are our cities, the most color-lifeful places you will ever see. This is a bit of a poetic-gradient description, but I do not know what else to say."`
```

			`	With these words the Quarg turns around and leaves. It seems like you are free to use the local facilities now.`

# TODO: Maybe add a version of the ships sprite that looks damaged/modded.
ship "Nimbus" "Nimbus FTL"
	add attributes
		"bunks" -25
	outfits
		"Liquid-Class Shield"
		"Metallic Hydrogen Cell"
		"Cloud Piercer" 20

		"MH Red-Class Engine"
		"MH Red-Class Steering"

mission "Incipias: The Derelict"
	landing
	invisible
	to offer
		has "Visit Quarg in Incipias Space: offered"
	npc assist
		government "Hicemus"
		ship "Nimbus FTL" "HIC Veni #000001"
		system "Hui'uc"
		to despawn
			has "incipias: helped the derelict"
		on assist
			conversation
				`The derelict ship hails you as you approach it. Once you accept the hail, color gradients start moving over your screen in a dizzying fashion - curiously, your speakers remain silent. It soon becomes clear that you will not be able to understand whatever transmission you just witnessed without some sort of translator. You continue your boarding maneuver.`
				scene "scene/incipias message"

> 				`	You manage to dock to the disabled ship quite easily. The hatch opens and a hallway opens up before you. It is immediately obvious that this ship was not designed for humans; the walls are strangely curved and the floor has nearly no grip at all. As you carry the necessary repair equipment into the ship, your view is caught by five giant aliens half octopode, half jellyfish in appearance. They pulsate in bright colors similar to the gradients that appeared on your screen earlier. One of them is floating behind a giant mounted gun, and the two barrels are pointed right at you. The gunner flashes in an orange light, and you can feel the tension in the room rising.`

Author: the previous reads more fluent to me, is it wrong from a grammar perspective?

> 				`	You manage to dock to the disabled ship quite easily. The hatch opens and a hallway opens up before you. It is immediately obvious that this ship was not designed for humans; the walls are strangely curved and the floor has nearly no grip at all. As you carry the necessary repair equipment into the ship, your view is caught by five giant aliens half octopode, half jellyfish in appearance. They pulsate in bright colors similar to the gradients that appeared on your screen earlier. One of them is floating behind a giant mounted gun, and the two barrels are pointed right at you. The gunner flashes in an orange light, and you can feel the tension in the room rising.`

Reviewer: Not really, it's more of a subjective preference imo.

				label end
				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment. Instead of leaving the system, however, the ship quietly disappears into the nearest gas giant.`
				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen. Notably this ship had no hyperdrive or anything similar installed; strange considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`
				action
					set "incipias: helped the derelict"
		personality
			confusion 50
			derelict uninterested staying
	on enter "Hui'uc"
		conversation
			`On entering the system you find an alien spaceship disabled and badly damaged. There do not appear to be any other ships or inhabited planets in the system.`

mission "Incipias: Help The Stranded 1"
	landing
	source
		government "Quarg (Incipias)"
	destination "Pon'tes"
	to offer
		has "incipias: helped the derelict"
	on offer
		conversation
			`Maybe the Quarg could help you with the stranded ship.`
			choice
				`Tell them about it.`
				`Not right now.`
					defer

> 			`	Right after you arrive on the planet, you search for a Quarg who speaks your language so that you can tell them about the stranded ship you found. One of the Quarg seems to understand your intention and gestures for you to wait. It returns a few hours later accompanied by the same Quarg you talked to before.`

Author: Nope just planets there

			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can foresee the likelihood of you making a rash and impetuous decision to contact this ship."`
			choice
				`	"Sure, let me do the work."`
				`	"If you are too weak..."`
			`	The Quarg rapidly clicks its teeth, before continuing.`
			label initiative
			`	"You lack the tools for such a mission. To my knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense."`
			`	"But for the sake of the Incipias, I will assist you on this last point. If you cannot be dissuaded from your course of action, then it is best you are supplied a means of communication, so that your interactions may be on amicable terms. This one can produce you with a single one of our translators, which will allow you to relate your intentions to the species in their home system. Request that they sell you one of their ships, so that you are made capable to land on their tumultuous planets."`

> 			`	It pauses before taking on a more serious tone. "However, you are forbidden from granting the young species any advanced technology of yours. If you concur with us on that point, we shall furnish you with a translator."`

Author: the engine does not provide any abillity for me to check that

mission "Incipias: Help The Stranded 2"
	landing
	passengers 5
	source
		planet Turra
	destination "Pon'tes"
	to offer
		has "Incipias: Help The Stranded 1: done"
	on offer
		conversation
			`Your ship dives into the clouds of the gas giant, entering an environment on a scale you never experienced before. Giant clouds tower like castles, sometimes opening to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`
			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and their shields are completely down - the shield generator must have gone offline when they ran out of fuel.`
			`	You receive no answer to your hail.`
			`	You carefully maneuver your ship closer, until you can dock with the Incipias ship.`

> 			`	When the lock opens five Incipias float in front of you. Noticing your presence they quickly back off; they were clearly not expecting a human in one of their species' ships. It takes a moment for them to recognize you.`

Author: hmm not sure what to do about that

> 			`	When the lock opens five Incipias float in front of you. Noticing your presence they quickly back off; they were clearly not expecting a human in one of their species' ships. It takes a moment for them to recognize you.`

Author: branches now

			`	You tell them to come aboard, and you manage to launch your ship just before a storm hits the area.`
				accept
	on complete
		conversation
			`When you land on the station your passengers thank you again and leave, probably to get some well-earned rest.`
			`	As you step out of the <ship> an Incipias, its scales seem worn off and you notice intricate claw appendices around the end of its tentacles, approaches you. "Follow me outer-helper, before a crowd gathers. We would like to talk."`
			`	They guide you through the hallways so that you avoid larger groups and move only through horizontal corridors. You arrive in a large room with a view over the entire station, with three Incipias already present. One begins to glow.`
			`	"Welcome. We are the leader-governers of the Hicemus, the reigning government of the Incipias. If you agree we would like to ask you a few question-queries. In return we would be open for question-queries as well."`
			choice
				`	"Sorry, but I have other stuff to do."`

> 					goto endNoTime

Author: should I not?

> 					goto endNoTime

Reviewer 7: > Why are you using camelCase?

It makes things easier to read, probably? :P

			choice
				`	"Why do you care about these 'flowers'?"`
					goto care
				`	"What I search for is not technology."`
			`	The voice now takes on a calm tone. "Your persistence on this topic is admirable, but allow me to explain my conviction with an example."`
			label care
			`	"Why do humans care about art?" The voice now seems far away yet more present than ever. "Because it amazes you, the same way it amazes us to see another kind grow and overcome their burdens. The lives of species are like a painting that never stop being painted, but when they do stop it is a tragic thing to watch. That is why we try to protect these creations of life - so we can see what heights they reach."`
			`	There is a pause. "I see that you can not be convinced by me. Even though I could simply wipe you out here and now, I do not favor violence." The voice is now resolute in tone. "I will let you pass, but I will be watching you, as I am the guardian of this space."`
			`	The voice fades out leaving nothing behind but the feeling of being watched.`
				decline
			label agree
			action

> 				set "Incipias: accepted the guardians warning"

Reviewer 8: ```suggestion
				set "Incipias: accepted the guardians warning"
				event "label incipias space"
```

				decline

mission "Incipias: Presence of the Guardian"
	invisible
	landing
	to complete
		never
	on enter Dom'us
		conversation
			`As you enter the system you have a feeling of being watched. The presence from the previous system still lingers.`
				to display
					not "Incipias: accepted the guardians warning"
			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." The voice fades away, but the strong presence remains.`
				to display
					has "Incipias: accepted the guardians warning"
				decline
	on enter Hui'uc
		conversation
			`As you enter the system you have a feeling of being watched. The presence from the previous system still lingers.`
				to display
					not "Incipias: accepted the guardians warning"
			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." The voice fades away, but the strong presence remains.`
				to display
					has "Incipias: accepted the guardians warning"
				decline

> mission "Discovered Incipias Space"
> 	landing
> 	invisible
> 	source
> 		government "Conlatio" "Hicemus" "Incipias Civilian" "Quarg (Incipias)"
> 	on offer
> 		event "label incipias space"
> 		fail

Reviewer 8: ```suggestion
```

# Copyright (c) 2023 by RisingLeaf (https://github.com/RisingLeaf)
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

> mission "Drak guarding Incipias Space"

Reviewer 6: ```suggestion
mission "Drak guarding Incipias space"
```
Capitalization is different in the below condition/this is how the other drak missions are capitalized. Also, side thought - this mission should probably be in the drak missions file; the other ones are (Remnant, Korath, etc.).

Author: moved it

			`	"Why do humans care about art?" The voice now seems far away yet more present than ever. "Because it amazes you, the same way it amazes us to see another kind grow and overcome their burdens. The lives of others are like a painting that never stops being painted, but is a tragic thing to watch the painting stop. That is why we try to protect these creations of life - so we can see what heights they reach."`
			`	There is a pause. "I see that you can not be convinced by me. Though I could intervene in your journey, I do not favor unsavory confrontation." The voice is now resolute in tone. "I will let you pass, but I will be watching you, as I am the guardian of this space."`
			`	The voice fades out leaving nothing behind but the feeling of being watched.`
				decline
			label agree
			action
				set "Incipias: accepted the guardians warning"
				event "label incipias space"
			`	"You are understanding, human, so I thank you in the name of the Incipias." The voice fades away, leaving only the feeling of being watched."`
				decline

mission "Incipias: Presence of the Guardian"
	invisible
	landing
	to complete
		never
	on enter Dom'us
		fail
		conversation

> 			`The presence from the previous system somehow lingers as you continue to feel as though you're being watched. `

Reviewer 6: ```suggestion
			`The presence from the previous system somehow lingers as you continue to feel as though you're being watched.`
```
Whoops.

Author: smh

				to display
					not "Incipias: accepted the guardians warning"
			`As you enter the system, the same voice from before echoes through your mind, though this time, there seems to be a heavy weight behind it. "I do not take kindly to your deception, human. Know that you will be watched." The voice fades away, but the strong presence of something remains.`
				to display
					has "Incipias: accepted the guardians warning"
				decline
	on enter Hui'uc
		fail
		conversation

> 			`The presence from the previous system somehow lingers as you continue to feel as though you're being watched. `

Reviewer 6: ```suggestion
			`The presence from the previous system somehow lingers as you continue to feel as though you're being watched.`
```

	on offer
		conversation
			`You identify the stranded ship on your scanners, but with no way to communicate with them you are unable to to help. You have no choice but to leave them to their doom on this stormy world.`

mission "Incipias: Help The Stranded 2"
	landing
	passengers 5
	source
		planet Turra
	destination "Pon'tes"
	to offer
		has "Incipias: Help The Stranded 1: done"
	on offer
		conversation
			`Your ship dives into the clouds of the gas giant, entering an environment on a scale you never experienced before. Giant clouds tower like castles, sometimes opening to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`
			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and their shields are completely down - the shield generator must have gone offline when they ran out of fuel.`
			`	You receive no answer to your hail.`

> 			`	You carefully maneuver your ship closer, until you can dock with the Incipias ship.`

Author: Would it not show up in the outfitter?

> 			`	You carefully maneuver your ship closer, until you can dock with the Incipias ship.`

Reviewer 6: If you buy a ship that has one, true, and if you were to go into the outfitter, if it appears there, yes. However, there's no guarantee the player does this or explicitly notes it, so I think "it must be the case" is a bit over-assuming versus "it could be the case".

Since this ship has been damaged in many ways, there's no telling if they ran out of fuel before or after their shields gave out; perhaps the storm damaged the device before they ran out of fuel.

So, saying "could" here is a bit more open-ended, and still acknowledges that the shield generator could stop working as a result of fuel outage.

				`	"I have been sent by your people to get you home."`
			`	They seem to switch from a tense state to a surprised one.`
			`	One of them starts pulsating, "You light our language? How surprising. We would not have survived another day here. You are our life-protector. We would love to talk more, but a weather-calamity is coming. We should leave now." The lack of emotion in the translator's voice is disconcerting given the content of the message.`
			`	You tell them to come aboard, and you manage to launch your ship just before a storm hits the area.`
				launch
	on complete
		conversation
			`When you land on the station your passengers thank you again and leave, probably to get some well-earned rest.`
			`	As you step out of the <ship> an Incipias approaches you. Its scales seem worn off and you notice intricate claw appendices around the end of its tentacles. "Follow me, outer-helper, before a crowd gathers. We would like to talk."`
			`	They guide you through the hallways so that you avoid larger groups and move only through horizontal corridors. You arrive in a large room with a view over the entire station, with three Incipias already present. One begins to glow.`

> 			`	"Welcome. We are the leader-governers of the Hicemus, the reigning government of the Incipias. If you agree we would like to ask you a few question-queries. In return we would be open for question-queries as well."`

Author: hmmm after reading the definition of both probably neither, it is supposed to be something to wear

> 			`	"Welcome. We are the leader-governers of the Hicemus, the reigning government of the Incipias. If you agree we would like to ask you a few question-queries. In return we would be open for question-queries as well."`

Reviewer 6: How about 'attachments'?

			choice
				`	"Sorry, but I have other stuff to do."`
					goto "end no time"
				`	"Sure, ask me something."`
			`	A green glow comes from all of them. "Thank you for taking the time-patience to speak with us. Can you tell us more about the deep space surrounding us?"`
			choice
				`	(Give them an exhausting story of what you have seen so far.)`
					goto detail
				`	(Give a rough overview of what you have seen.)`
			`	After seeing your explanation a rapid-fire sequence of colors are exchanged between them. Your translator cannot follow, so you are left watching them flash in yellow tones mixed with green and red.`

> 			`	"Thank you for this insight. All we gathered until now was by observing-studying stars. To think that there is so much out there... it truly raise-strengthens a determination to explore.`

Author: well they would "see" an explanation, it is another hint at their communication quirk

> 			`	"Thank you for this insight. All we gathered until now was by observing-studying stars. To think that there is so much out there... it truly raise-strengthens a determination to explore.`

Reviewer 6: True. What if we just put quotes around it then? After they "see" your explanation....

> 			`	"Thank you for this insight. All we gathered until now was by observing-studying stars. To think that there is so much out there... it truly raise-strengthens a determination to explore.`

Author: ye that works

> 			`	"Thank you for this insight. All we gathered until now was by observing-studying stars. To think that there is so much out there... it truly raise-strengthens a determination to explore.`

Reviewer 6: Updated both suggestions.

				goto yours
			label detail

> 			`	After watching your explanation a rapid-fire sequence of colors are exchanged between them. Your translator cannot follow, so you are left watching them flash in yellow tones mixed with green.`
> 			`	"Thank you for this profound-depth insight. All we gathered until now was by observing-studying stars close to us. To think that there is so much out there... it truly invokes a determination to explore.`

Reviewer 6: ```suggestion
			`	After being given your explanation, a rapid-fire sequence of colors is exchanged between the Incipias. Your translator cannot follow, so you are left watching them flash in what ends up being yellow tones mixed with green.`
			`	"Thank you for this profound-depth insight. All we gathered until now was by observing-studying stars close to us. To think that there is so much out there... it truly invokes a determination to explore.`
```

			label yours
			`	"But now to you. Is there anything you would like to know?"`
			label choice
			choice

> 				`	"I do not have any more questions."`
> 					goto end
> 				`	"Why do you call this station your capital?"`
> 					goto capital
> 				`	"How is life on a gas giant?"`
> 					goto oddquestion
> 				`	"Why can I not land on your home planet?"`

Reviewer 6: ```suggestion
				`	"Why can I not land on your home planet?"`
				`	"Why do you call this station your capital?"`
					goto capital
				`	"How is life on a gas giant?"`
					goto oddquestion
				`	"I do not have any more questions."`
					goto end
```
Placing the end at the bottom option. I think overall this order is better too.

			`	A flash of red appears for only a second. "Our origin-world is not directly ruled by us. They may approach you separately at a future-point, but be careful of their lights. They have aged-time values long left-alone by us, who are progressing fast.`
				goto choice
			label capital
			`	The one pulsating now takes on a bright green color, "This station is our greatest action-pride, our star-bridge. In honor of our urge to expand we made this station our capital to demonstrate our determination."`
				goto choice
			label oddquestion

> 			`	"What an odd-reasonable question," it turns into a swirl of purple and green, "We could ask the same in return, what is it like to live on hard rock? But we understand its reasoning-origin. So let me tell you of the cloud-halls of our home-sky. Lightning will draw the most beautiful color-sings between the pillar-clouds between them are our cities, the most color-liveful places you will ever see. This is a bit poetic-gradient description but I do not know what else to say."`

Reviewer 6: ```suggestion
			`	"What an odd-reasonable question." It's glow turns into a swirl of purple and green. "We could ask the same in return, what is it like to live on hard rock? But we understand its reasoning-origin. So let me tell you of the cloud-halls of our home-sky. Lightning will draw the most beautiful color-sings between the pillar-clouds between them are our cities, the most color-lifeful places you will ever see. This is a bit poetic-gradient description but I do not know what else to say."`
```

			label end
			`	"Thank you again for your time. You have given us much to process. We hope you too have benefited from this meeting-talk."`

> 			`	You say goodbye and as you leave the room you can see a rush of colors reflecting on the walls.`

Reviewer 6: ```suggestion
			`	You say goodbye, and as you leave the room, you can see a rush of colors reflecting onto the walls.`
```

			`	There is a pause. "I see that you can not be convinced by me. Even though I could simply wipe you out here and now, I do not favor violence." The voice is now resolute in tone. "I will let you pass, but I will be watching you, as I am the guardian of this space."`
			`	The voice fades out leaving nothing behind but the feeling of being watched.`
				decline
			label agree
			action
				set "Incipias: accepted the guardians warning"
				event "label incipias space"
			`	"You are understanding, human, so I thank you in the name of the Incipias." The voice fades away, leaving only the feeling of being watched."`
				decline

mission "Incipias: Presence of the Guardian"
	invisible
	landing
	to complete
		never
	on enter Dom'us
		fail
		conversation
			`As you enter the system you have a feeling of being watched. The presence from the previous system still lingers.`
				to display
					not "Incipias: accepted the guardians warning"

> 			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." The voice fades away, but the strong presence remains.`

Author: well the player did say they were going to leave the space, so it is a lie

> 			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." The voice fades away, but the strong presence remains.`

Reviewer 6: >well the player did say they were going to leave the space, so it is a lie

This is only if you click the option that says you will leave the systems. You can still choose to answer differently without heeding the Archon's warning and continue into the systems, in which case it's not deception, just ignorance of what it said.

> 			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." The voice fades away, but the strong presence remains.`

Reviewer 6: Also just tested; even if I heed the warning and am an understanding human, I still get the dialog that im deceiving it.

> 			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." The voice fades away, but the strong presence remains.`

Author: no problem

				to display
					not "Incipias: accepted the guardians warning"
			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." The voice fades away, but the strong presence remains.`
				to display
					has "Incipias: accepted the guardians warning"
				decline

event "label incipias space"
	galaxy "label incipias"
		sprite "label/incipias"

mission "Incipias: First Contact"
	invisible
	landing
	to complete
		never
	on enter Dom'us
		conversation
			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. Maybe you can find some more information in nearby systems.`
				to display
					not "Visit Quarg in Incipias Space: offered"
			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. This must be the species the Quarg talked about.`
				to display

> 					has "Visit Quarg in Incipias Space: offered"

Reviewer 6: I'm not sure these really need to describe the player hailing the ships. They would do so curiously outside of telling them; I think it's better to leave this action up to the player performing it themselves rather than tell them they did it before they even try to in-game.

If the purpose is to lead into the line to explore other nearby systems, I also don't think that's strictly necessary - this is a brand new area, the player will explore eventually find the Quarg anyways.

Though I'm not sure of a good alternative at the moment.

Author: Hm how about they hail you?

Author: "speakers stay silent but a colorful haze appears on your screen." smth like that

Reviewer 6: I feel saying they hail you here would take away from the "first direct hail" you get with the scene. It could just be that they don't try to hail you initially, and that's what you say. Perhaps they're hesitant to start with.

			`	You have to wait for a few hours until a group of three Quarg appear. The one in the middle begins to speak in an accent you can't place.`
			label "both planets"
			`	"Welcome, human. Your lengthy wanderings have brought forth you here, yet your kind has never set foot in this region of the stars. May we inquire as to your mission?"`
			choice
				`	"I am just exploring the galaxy."`
					goto explorer
				`	"I am looking for new species."`
					goto "new species"
			label explorer
			`	"It is an obvious pursuit for the young," it nods. "If I may advise you, do not perambulate across this quarter of space in a careless manner. The galaxy carries many perilous situations that your kind has yet to uncover or even to comprehend." It straightens up to its full height, towering over you.`
				goto end
			label "new species"
			`	"What a peculiar task that is." It looks confused and tilts its head. "What do you hope to achieve by this? That an unfathomable amount of dangers come with encountering and treating with other species, did you not foresee? The boundaries of space are not idle accidents. I advise you to be scrupulous and ponder every last step."`
			label end

> 			`	After a short pause it continues talking; you notice an almost-hidden frustration lurking beneath the Quarg's face. "You are authorized to continue your nomadic voyagings here, but there are demands we Quarg make of you. Within a trifling interval of this moon another kind has initiated their journey to the stars. Do not interrupt them for your own sake, as we will avenge any hostile actions against them."`

Reviewer: I was the one that added 'voyagings', to suggest that this Quarg doesn't have a very good grasp of humanese.

> 			`	After a short pause it continues talking; you notice an almost-hidden frustration lurking beneath the Quarg's face. "You are authorized to continue your nomadic voyagings here, but there are demands we Quarg make of you. Within a trifling interval of this moon another kind has initiated their journey to the stars. Do not interrupt them for your own sake, as we will avenge any hostile actions against them."`

Author: I think most people will understand what the word is supposed to mean?

> 			`	After a short pause it continues talking; you notice an almost-hidden frustration lurking beneath the Quarg's face. "You are authorized to continue your nomadic voyagings here, but there are demands we Quarg make of you. Within a trifling interval of this moon another kind has initiated their journey to the stars. Do not interrupt them for your own sake, as we will avenge any hostile actions against them."`

Reviewer 6: Fair enough. Changed voyagings back; the rest of the suggestion still applies.

> 			`	After a short pause it continues talking; you notice an almost-hidden frustration lurking beneath the Quarg's face. "You are authorized to continue your nomadic voyagings here, but there are demands we Quarg make of you. Within a trifling interval of this moon another kind has initiated their journey to the stars. Do not interrupt them for your own sake, as we will avenge any hostile actions against them."`

Reviewer 6: I mean, if one is reading the conversation, then they'd see my comment later saying it's okay; it would be odd to just delete something in a prior comment when other comments exist based on that discussion.

				choice
					`	(Try to convey that I intend no harm.)`
						goto peace
					`	(Drop the repair equipment and run back into my ship.)`
						goto flee
					`	(Try to make a non-lethal shot at the one behind the turret and run.)`
				`	You hit the creature manning the turret, causing it to explode with a strident sound. Before you can enjoy your short-lived victory, you notice a motion further down the hallway. Turning your head you see a gunner flashing in a deep red, before shooting you down and avenging their fallen comrade.`
					die
				label peace

> 				`	Feeling that the situation could escalate any moment, you slowly place the crate with the repair equipment on the ground and walk back a few steps. You are careful not to make any hasty movements. Both sides are still for a moment until one of the aliens slowly floats forward. Tones of yellow are now mixing into the creature's orange color, and two of their tentacles wrap around the crate. The other four back off in response. Once the alien manages to open the crate and look inside, its colors shift to a weak green. The rest of the crew also seem to relax as they take on the same color.`

Reviewer 6: ```suggestion
				`	Feeling that the situation could escalate any moment, you slowly place the crate with the repair equipment on the ground and walk back a few steps. You are careful not to make any hasty movements. Both you and the aliens are still for a moment until one of them slowly floats forward. Tones of yellow are now mixing into the creature's orange color, and two of their tentacles wrap around the crate while the other four back off in response. Once the alien manages to open the crate and look inside, its colors shift to a weak green. The rest of the crew also seem to relax as they take on the same color.`
```
Slight rewording.

> 				`	With the repair equipment delivered, there is nothing else you can do, given the communication barrier. Just as you leave you can see how the aliens start pulsating in all colors of the spectrum; gradients are moving over their skin so fast that they almost look white.`

Reviewer 6: ```suggestion
				`	With the aliens retrieving the repair equipment, there is nothing else you can do, given the communication barrier. Just as you leave, you can see how the aliens start pulsating in all colors of the spectrum; gradients are moving over their skin so fast that they almost look white.`
```
This wasn't really a "delivery"; you wanted to help them, got scared, and then left the equipment.

					goto end
				label flee
				`	You drop the repair equipment and run back into your ship, smashing the button that closes the lock. You get a last look of one of the aliens: blue and yellow tones are mixing on their skin, but they are not taking any action.`
				label end

> 				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment. Instead of leaving the system, however, the ship quietly disappears into the nearest gas giant.`
> 				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen. Notably this ship had no hyperdrive or anything similar installed; strange, considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`

Reviewer 6: ```suggestion
				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment. Instead of leaving the system, however, the ship begins to fly towards a nearby gas giant.`
				`	Back on your ship, your eyes fall on the scanner log that is still open on your monitor. Notably, this ship had no hyperdrive or anything similar installed, which is strange considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they have any insight on this.`
```
Since the ship doesn't actually land right now, this is a better alternative. Also, your ship could be a fighter or medium-sized ship, nothing guarantees you having a bridge as opposed to a cramp cockpit. Instead of ask the Quarg for help, considering you did just help the aliens, it makes more sense to ask them for information/insight here.

				action
					set "incipias: helped the derelict"
		personality
			confusion 50
			derelict staying uninterested
	on enter "Hui'uc"
		conversation
			`On entering the system you find an alien spaceship disabled and badly damaged. There do not appear to be any other ships or inhabited planets in the system.`

mission "Incipias: Help The Stranded 1"
	landing
	source
		government "Quarg (Incipias)"
	destination "Pon'tes"
	to offer
		has "incipias: helped the derelict"
	on offer

> 		conversation
> 			`Maybe the Quarg could help you with the stranded ship.`
> 			choice
> 				`	(Tell them about it.)`
> 				`	(Not right now.)`

Reviewer 6: ```suggestion
		conversation
			`Would you like to ask the Quarg for insight on your strange encounter with the stranded alien ship?`
			choice
				`	(Yes.)`
				`	(Not right now.)`
```
In other places in the game, if we have a short defer dialog, we typically ask the player in a form of a question if they want to continue. For example, Hai first contact.

					defer

> 			`	Right after you arrive on the planet, you search for a Quarg who speaks your language so that you can tell them about the stranded ship you found. One of the Quarg seems to understand your intention and gestures for you to wait. It returns a few hours later accompanied by the same Quarg you talked to before.`

Reviewer 6: ```suggestion
			`	Right after you arrive on the planet, you search for a Quarg who speaks your language so that you can tell them about the stranded ship you found. One of the Quarg seems to understand your intention and gestures for you to wait. It returns a few moments later accompanied by the same Quarg you talked to before.`
```
Make this a bit more open-ended; we do that elsewhere for smaller encounters like this (the player probably isn't going stand/sit there motionless for hours on end). Also especially when right below this, the Quarg acknowledges haste.

			`	"Human, you seem to be in quite a haste. What could be such a pressing matter?" it asks.`
			choice
				`	(Explain the situation.)`
					goto explanation
				`	"I need advice regarding the species in this region."`
			`	It looks at you curiously. "So you have held council with the young Incipias? This one is nonplussed that you ably formed correspondence. It took a prolonged period of time even for us to decipher their way of dissemination."`
			`	You quickly explain what happened. From what you can tell, the Quarg seems relieved that you were not able to talk to them.`
			label explanation
			`	"You ran into a most peculiar situation. Surpassing the borders of their system is not yet a capability possessed by the Incipias." It tilts its head for a moment, considering what you just said. "It is our desire to help them, but it is yet premature for us to reveal ourselves to their species, and certainly premature for them to be interacting with independent interlopers such as yourself. I fear that we are impotent in this unfortunate turn of events."`
			choice

> 				`	"Then let me help them."`

Reviewer 6: ```suggestion
				`	"I could help them."`
```
This early in the conversation it'd be better to suggest you can help them, and then later on be more pressing about letting you do so.

					goto initiative
				`	"I understand. I'll leave them alone."`
					goto declined
				`	"How can you make such a cold-hearted decision?"`
			`	"It is perhaps a laborious task for an adolescent species such as yourself to comprehend our ways, but we must place the continuation of species above the existence of individuals. If we reveal ourselves to them at this inopportune moment, the shock of such an advanced species appearing in their space will fan the flames of chaos."`
			choice
				`	"Then let me help them."`
					goto initiative
				`	"You can deal with the consequences later. Help them now!"`

> 			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can foresee the likelihood of you making a rash and impetuous decision to contact this ship."`

Reviewer 6: ```suggestion
			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, this one can foresee the likelihood of you making a rash and impetuous decision to contact these Incipias further."`
```
Consistency of "this one" vs "I". Also, you already contacted the ship, so contacting it further beyond that would be the concern.

			choice

> 				`	"Sure, let me do the work."`

Reviewer 6: ```suggestion
				`	"Sure. Let me do the work."`
```

> 				`	"If you are too weak..."`

Reviewer 6: ```suggestion
				`	"If you insist on not helping..."`
```
Less combativeness and more objective on what the Quarg won't do (help)/

> 			`	The Quarg rapidly clicks its teeth, before continuing.`

Reviewer 6: ```suggestion
			`	The Quarg rapidly clicks its teeth before continuing.`
```
Unnecessary comma.

> 			label initiative
> 			`	"You lack the tools for such a mission. To my knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense."`
> 			`	"But for the sake of the Incipias, I will assist you on this last point. If you cannot be dissuaded from your course of action, then it is best you are supplied a means of communication, so that your interactions may be on amicable terms. This one can produce you with a single one of our translators, which will allow you to relate your intentions to the species in their home system. Request that they sell you one of their ships, so that you are made capable to land on their tumultuous planets."`
> 			`	It pauses before taking on a more serious tone. "However, you are forbidden from granting the young species any advanced technology of yours. If you concur with us on that point, we shall furnish you with a translator."`

Reviewer 6: ```suggestion
			label initiative
			`	"You lack the tools for such a mission. To my knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense."`
			`	"But for the sake of the Incipias, I will assist you on this last point. If you cannot be dissuaded from your course of action, then it is best you are supplied a means of communication, so that your interactions may be on amicable terms. This one can produce you with a single one of our translators, which will allow you to relate your intentions to the species in their home system. Request that they sell you one of their ships, so that you are made capable to land on their tumultuous planets."`
			`	It pauses before taking on a more serious tone. "However, you are forbidden from granting the young species any advanced technology of yours. If you concur with us on that point, we shall furnish you with a translator."`
```
```suggestion
			label initiative
			`	"You lack the tools for such a mission. To this one's knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense." The Quarg stares at you for a moment, pondering a little.`
			`	"But for the sake of the Incipias, this one will assist you on this last point. If you cannot be dissuaded from your course of action, then it is best you are supplied a means of communication, so that your interactions may be on amicable terms. This one can produce you with a single one of our translators, which will allow you to relate your intentions to the species. Request that they sell you one of their ships, so that you are made capable to land on their tumultuous planets."`
			`	It pauses before taking on a more serious tone. "However, you are forbidden from granting the young species any advanced technology of yours. If you concur with us on that point, we shall furnish you with a translator."`
```
Consistency with "this one", and added a small narration at the end of the first big paragraph. If there's two dialogs of quotes and they're split into paragraphs, it's good to have something narrative happen, otherwise it is just a long quotation that could be one quote.

You don't need to specify their home system because, well, that's the only place their kind will be, and the only place they sell their ships.

			label choice
			choice
				`	"If that's how I can help them, sure."`
					goto yes

> 				`	"Why do you not want me to sell my tech?"`

Reviewer 6: ```suggestion
				`	"Why do you not want me to give them my tech?"`
```
You might not sell them, you might just give them a Gridfire turret for fun for all the Quarg know.

			`	"Halting the natural increase of a species is unpredictable at the best of times. Your species lacks the wisdom to understand this, as you are young, and have not seen what we have seen." The Quarg pauses for a moment, lost in thought. "The foreseeable risks of even letting you contact them are monumental, and not something you should consider trivial."`
				goto choice
			label yes

> 			`	"Happy I am that you agree to that condition. Bear in mind, Captain, that the slightest misstep may have grave consequences." The Quarg stares you down. "We will be watching your actions, to verify that they are peaceful and without ill or unwise intent. Please for a moment remain here," it says, and walks away.`
> 			`	The Quarg only takes a moment to come back; it is now carrying a sleek silver box with a display on one side.`
> 			`	"This is our translator, our link to the Incipias. It is able to translate their unwonted language into your speech as well. Be aware of the honor of me gifting this to you," it says.`
> 			`	As you stow it away, the Quarg speaks once more. "To satisfactorily utilize this device, you must direct it towards the source of the communication, so that it is made able to discern the relevant emissions. Given the complexity of their language, the translator might not be able to adequately translate some aspects of their obscure speech."`
> 			`	It then looks you directly in the eyes. "Remember what I have told you, human. The stars have no place for those who would act foolishly and thoughtlessly towards their fellow beings." The Quarg then walks away, leaving you alone.`

Reviewer 6: ```suggestion
			`	"Happy this one is that you agree to that condition. Bear in mind, Captain, that the slightest misstep may have grave consequences." The Quarg stares you down. "We will be watching your actions, to verify that they are peaceful and without ill or unwise intent. Please for a moment remain here," it says, and walks away.`
			`	The Quarg only takes a moment to come back, now carrying a sleek silver box with a display on one side. "This is our translator, our link to the Incipias. It is able to translate their unusual language into your speech as well. Be aware of the honor of this one gifting this to you," it says.`
			`	As you stow it away, the Quarg speaks once more. "To satisfactorily utilize this device, you must direct it towards the source of the communication, so that it is made able to discern the relevant emissions. Given the complexity of their language, the translator might not be able to adequately translate some aspects of their obscure speech."`
			`	It then looks you directly in the eyes. "Remember what has been told, human. The stars have no place for those who would act foolishly and thoughtlessly towards their fellow beings." The Quarg then walks away, leaving you alone.`
```
Combined two of the smaller indented sentences into a paragraph, and changed self-references to "this one" for consistency. Also changed unwonted to unusual (unwonted is too odd of a world here and can read strangely).

				accept
			label declined

> 			`	"We are gracious for your understanding, but remember thus: the Incipias are under our watch and care. None shall come to harm them without incurring our wrath." With these words the Quarg turns away.`

Reviewer 6: ```suggestion
			`	"We are gracious for your understanding, but remember thus: the Incipias are under our watch and care. None shall come to harm them without incurring our wrath." With these words the Quarg turns away.`
```
Just commenting that I feel this is a better place for them to be a bit more on the nose about intervening, and since I suggested prior that the Archon says intervene instead of 'wipe you out', it makes sense here that the Quarg are a bit more direct.

				decline
	on enter Dom'us
		"reputation: Hicemus" += 10
		conversation

> 			`Entering the system with the translator, you prepare to broadcast your first message.`

Reviewer 6: ```suggestion
			`After you enter the system, you prepare to broadcast your first message with the Quarg's translator.`
```
A bit more of a narrative lead-in than "entered place, you do thing".

			choice
				`	"Hello. I am Captain <first> <last>, here to make contact."`
				`	"Greetings, strangers. I seek to help you."`
				`	"I come in peace."`

> 			`	You notice that your translator starts to flash a sequence of lights, and you point it towards your ship's camera. For a minute, nothing happens at all.`

Reviewer 6: ```suggestion
			`	You notice that the translator starts to flash a sequence of lights, and you point it towards your ship's communication device. For a minute, nothing happens at all.`
```
It's not *yours*, it's the Quargs. Changed it to simply "the".
With a lot of different ships and factions and such, it's better to keep the internal devices obscure. In fact, you probably wouldn't point it at a camera anyways - it's a magical Quarg device, it probably links up directly to your monitors.

> 			`	After some time, a wave of movement passes through all ships in the system, and a communication channel opens from the space station. Colors rush over your display: a mix of yellow and green. You point your translator towards the flashing lights and hope it is able to pick up whatever these aliens are saying to you.`
> 			`	"Welcome world-traveler. We greet you in our home-space." The voice is smooth and synthetic, lacking in any discernible emotion.`
> 			`	The colors stop before rushing over your screen again.`
> 			`	"What is your intention-goal here?"`

Reviewer 6: ```suggestion
			`	After some time, a wave of movement passes through all ships in the system, and a communication channel opens from the space station. Colors rush over your display: a mix of yellow and green. You point your translator towards the flashing lights and hope it is able to pick up whatever these aliens are saying to you.`
			`	"Welcome world-traveler. We greet you in our home-space." The voice is smooth and synthetic, lacking in any discernible emotion. The colors stop before rushing over your screen again. "What is your intention-goal here?"`
```
This can just be all one paragraph; it's odd to have three split lines like this in a row.

			choice
				`	"I am here to make peaceful contact with you."`

> 				`	"I want to help one of your stranded ships."`
> 			`	Again a long pause. "Your intention-goals are welcome. Please dock to the capital-station."`
> 			`	Along the message comes a small map of the station with the docking ports highlighted.`

Reviewer 6: ```suggestion
				`	"I want to help one of your stranded ships."`
			`	Again a long pause. "Your intention-goals are welcome. Please dock to the capital-station." Along the message comes a small map of the station with the docking ports highlighted.`
```
Ditto; unless a choice is splitting up short sentences, there's no need to do it otherwise.

	on complete
		conversation

> 			`Thanks to the map, you are easily able to dock with the station. Your airlock opens and you enter an environment like nothing you ever saw before. There are no hallways like you would see in a human space station - instead, tubes full of fog, some of which even ascend vertically, appear to be the only way of getting around. A crowd of beings gathered in front of your airlock, each flashing a multitude of colors. A group of five floats in front of the crowd, and the middle one starts pulsating in calm tones.`

Reviewer 6: ```suggestion
			`Thanks to the map, you are easily able to dock with the station. Your airlock opens and you enter an environment like nothing you've ever seen before. There are no hallways like you would see in a human space station - instead, tubes full of fog, some of which even ascend vertically, appear to be the only way of getting around. A crowd of beings gathered in front of your airlock, each flashing a multitude of colors. A group of five floats in front of the crowd, and the middle one starts pulsating in calm tones.`
```

> 			`	"Welcome, outer-being, to our capital. We are the Incipias."`
> 			`	It stops as if awaiting an answer.`

Reviewer 6: ```suggestion
			`	"Welcome, outer-being, to our capital. We are the Incipias." It stops as if awaiting an answer.`
```
This is a better line to have separated from the paragraph before the choice, but we don't need to separate into two even smaller indents.

			choice
				`	"Greetings, Incipias. I am a human."`
				`	"I thank you for your hospitality."`
			`	Your translator begins rapidly flashing. The Incipias seem to respond to this - movement and colors begin to diffuse through the crowd like waves.`
			`	The 'speaker' continues. "You are the first outer-being that ever communicated with us. Are you the only outer-being out there?"`
			choice
				`	"No, there are others too."`
				`	"The galaxy is full of other beings."`

> 			`	Another wave goes through the crowd. "So there are more. But, stranger, how can we help you?"`

Reviewer 6: ```suggestion
			`	Another wave goes through the crowd. "So there are more. But, far-stranger, how can we help you?"`
```
Just so there's a translation bit here. It could also be distant-stranger or whatever else you feel works best.
